<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>PSI spring workshop 2004</title>
</head>
<body>
<h3 style="text-align: center;"><a name="mozTocId830364"
 class="mozTocH3"></a><a href="http://psidev.sourceforge.net"><img
 alt="Proteomics Standards Initiative" src="../../images/psi.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="left"
 title=""></a> <a href="http://www.hupo.org/"><img
 alt="HUPO Human Proteome Organisation" src="../../images/hupo.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="right"
 title=""></a>HUPO Proteomics Standards Initiative <br>
</h3>
<h3 style="text-align: center;"><a name="mozTocId313157"
 class="mozTocH3"></a>Spring Workshop <br>
</h3>
<h3 style="text-align: center;"><a name="mozTocId689138"
 class="mozTocH3"></a>April 19-21, 2004</h3>
<h3 style="text-align: center;"><a name="mozTocId528852"
 class="mozTocH3"></a>Carros, near Nice, Cote d'Azur, France<br>
</h3>
<h3>Contents</h3>
<ol id="mozToc">
  <li><a href="#intro">Introduction</a></li>
  <li><a href="#structure">Meeting structure</a></li>
  <li><a href="#agenda">Detailed agenda</a></li>
  <li><a href="#logistics">Logistics</a></li>
  <li><a href="#acknowledgements">Sponsors</a></li>
</ol>
<ol id="mozToc">
<!--mozToc h3 1-->
</ol>
<h3><a name="intro" class="mozTocH3"></a>Introduction</h3>
Since its conception in April 2002, the HUPO Proteomics Standards
Initiative has contributed to the development of community standards
for proteomics in a collaborative and very dynamic way, resulting in
several <a href="http://psidev.sourceforge.net/#publications">published
</a><a href="http://psidev.sourceforge.net/#publications">reports</a>.
The PSI Molecular Interaction standard has been <a
 href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14755292&amp;dopt=Abstract">published</a>
in Nature Biotechnology in February 2004, and the PSI Mass Spectrometry
standard is being finalised and already supported by a comprehensive
set of tools. The initial strategy of focussing on limited, feasible
domains
has been rewarded by significant progress in the focal domains within
only 18 months.<br>
<br>
However, a lot remains to be done. Building on the successful
development and quick adoption of the PSI MI Level 1 standard, we now
need to exploit its potential by increasing inter-database
collaboration to maximise database interoperability, and to promote it
by contacting journals and funding agencies and asking them for support
for the standard. <br>
The PSI Mass Spectrometry work group aims to extend its coverage beyond
the peak list to encompass the full chain from the mass spectrometer to
the identification list. Building on this set of standards, strategies
for its publication and acceptance need to be developed.<br>
And finally, we need to intensively work on the integration of
technology- or domain-specific standards into an encompassing standard
for proteomics data, similar to the work of, and in collaboration with
the MGED consortium for the transcriptomics domain.<br>
<br>
We will address these issues a the PSI Spring Workshop, April
19-21, Carros, Nice, Cote d'Azur, France. Participation in the meeting
is free, thanks to generous <a href="#acknowledgements">sponsorship</a>.<br>
<h3><a name="structure" class="mozTocH3"></a>Meeting structure<br>
</h3>
<ul>
  <li>
    <table cellpadding="2" cellspacing="4" border="1"
 style="width: 100%; text-align: left; background-color: rgb(255, 255, 255);">
      <tbody>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);"><br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Monday,
19-Apr-2004<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Tuesday,
20-Apr-2004<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Wednesday,
21-Apr-2004<br>
          </td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">9-12<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(255, 255, 255);"
 rowspan="1" colspan="1">Arrival and registration<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);"
 rowspan="1" colspan="1"> Work groups:<br>
          <ul>
            <li>Molecular Interactions: <br>
              <span style="font-style: italic;">Salle Mimosa<br>
              </span><a href="#mi-detailed-agenda">(detailed agenda
below)</a><span style="font-style: italic;"><br>
              </span></li>
            <li>Miape: <span style="font-style: italic;">Salle Hibiscus</span><br>
              <strong style="font-weight: normal;">- Short talks series</strong>
-<br>
              <a href="#miape-detailed-agenda">(detailed agenda below)</a><br>
            </li>
            <li>Mass spectrometry: <br>
              <span style="font-style: italic;">Salle Hibiscus<br>
              </span><a href="#ms-detailed-agenda">(detailed agenda
below)</a><br>
            </li>
          </ul>
          <br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">9:00-10:00:
          <br>
          <ul>
            <li>Work groups summary, writeup</li>
          </ul>
          <br>
10:00-10:15: Coffee<br>
          <br>
10:15: Plenary session: <span style="font-style: italic;">Salle
Hibiscus</span><br>
          <ul>
            <li>Work group progress reports<br>
            </li>
          </ul>
          <ul>
            <li>publications and dissemination</li>
            <li>Planning for HUPO meeting Beijing, October 2004</li>
            <li>closing remarks<br>
            </li>
          </ul>
          </td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">12-14<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Buffet
lunch<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);"
 rowspan="1">Buffet lunch<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Buffet
lunch<br>
          </td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">14-18
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Plenary
session: <span style="font-style: italic;">Salle Hibiscus</span>:<br>
          <ul>
            <li>Welcome; <br>
Molecular Interactions group:<br>
Henning Hermjakob</li>
            <li>Mass spectrometry group: <br>
Randall Julian/Weimin Zhu</li>
            <li>Miape work group:<br>
Chris Taylor</li>
          </ul>
          <br>
16:00-16:30 <br>
          <ul>
            <li>Coffee break</li>
          </ul>
          <br>
Pleanary session:<br>
          <ul>
            <li>The MGED standardisation process:<br>
Angel Pizarro</li>
          </ul>
          <br>
Split into work groups:<br>
          <ul>
            <li>Molecular interactions: <span
 style="font-style: italic;">Salle Mimosa</span></li>
            <li>mzData: <span style="font-style: italic;">Salle
Hibiscus</span></li>
            <li>Miape: <span style="font-style: italic;">Salle Hibiscus</span><br>
            </li>
          </ul>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Work
groups:<br>
          <ul>
            <li>Molecular Interactions: <span
 style="font-style: italic;">Salle Mimosa</span><br>
            </li>
            <li>Mass spectrometry: <br>
              <span style="font-style: italic;">Salle Hibiscus</span><br>
            </li>
            <li>Miape: <span style="font-style: italic;">Salle Hibiscus</span><br>
            </li>
          </ul>
          <br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(255, 255, 255);"
 rowspan="2" colspan="1">Departures<br>
          </td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">18.30<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Workshop
dinner:<br>
          <ul>
            <li>18:30: Busses depart from Kyriad Hotel <br>
and go to Carros Village. <br>
Free time to explore medieval Carros.</li>
            <li>20:00: Restaurant La Forge</li>
            <li>23:00: Busses return to hotel<br>
            </li>
          </ul>
          </td>
          <td style="vertical-align: top;">Free, restaurant suggestions
will be provided<br>
          </td>
        </tr>
      </tbody>
    </table>
    <br>
  </li>
</ul>
<br>
<h3><a name="agenda" class="mozTocH3"></a>Detailed agenda<br>
</h3>
This is an open, draft agenda. Please submit both agenda
items and suggested short presentations to <a
 href="mailto:hhe@ebi.ac.uk">hhe@ebi.ac.uk</a><br>
<ul>
  <li>Parallel work groups:<br>
  </li>
  <ul>
    <li>
      <h3><a name="ms-detailed-agenda"></a>Mass Spectrometry (Weimin
Zhu)</h3>
    </li>
    <ol>
    </ol>
    <ul>
      <li>Future development plan</li>
      <li>Controlled vocabulary for M/Z data exchange format</li>
      <li>Converters to convert supported vendor proprietary peaklist
formats to</li>
      <ul>
        <li>standard XML format.</li>
        <li>standard XML format for search engine results</li>
      </ul>
      <li>Search engines to be supported</li>
      <li>XML schema</li>
      <li>Tools: navigation, visulization</li>
      <li>How to adapt PSI-MS standard data formats</li>
      <li>PSI-MS data formats and MIAPE data model.</li>
    </ul>
    <li>
      <h3><a name="mi-detailed-agenda"></a>Molecular Interactions
(Henning Hermjakob)</h3>
    </li>
    <ol>
    </ol>
    <ul>
      <li>Status overview and plan</li>
      <li>Short presentations on PSI MI support, tools, and
expectations:</li>
      <ul>
        <li>Volker Stuempflen, MIPS, Germany:<span
 style="font-family: monospace;"> </span>MIPS Protein Interaction
Databases and the Integration of the PSI MI Standard</li>
        <li>Henning Hermjakob, EBI, UK: IntAct static and dynamic PSI
data download</li>
        <li>Gary Bader, MSKCC, New York, US: BioPax and Cytoscape</li>
        <li>Luisa Montecchi, MINT, Rome, Italy: Mint PSI MI tools<br>
        </li>
      </ul>
      <li>From PSI MI 1.0 to 1.1:<br>
Updates to the schema to fix problems which have shown up during the
use in production mode. <br>
      </li>
      <ul>
        <li>Schema updates.<br>
For a detailed listing of items see<br>
          <a
 href="http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse">http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse</a></li>
      </ul>
      <ul>
        <li>Controlled vocabulary updates and additions<br>
        </li>
      </ul>
      <ul>
        <ul>
          <li>Shared controlled vocabulary for database crossreferences<br>
          </li>
        </ul>
      </ul>
      <li>From PSI MI 1.1 to 1.9:</li>
      <ul>
        <li>Larger updates which extend the capabilities of the PSI MI
schema, e.g. addition of RNA as interactor.</li>
        <li>Additional controlled vocabularies, major extensions:</li>
      </ul>
      <ul>
        <ul>
          <li>interactionType</li>
        </ul>
      </ul>
      <li>Unification of identifiers<br>
Currently it is hard to detect reduncancy in and between PSI files, as
database xrefs and identifiers for objects are not standardised.<br>
      </li>
      <li>Formal organisation for the maintanance of PSI MI schema and
vocabularies<br>
      </li>
      <li>PSI MI based data exchange:<br>
      </li>
      <ul>
        <li>Agreement to make all PSI submitted data available to all
databases and end users</li>
        <li>Approach publishers to make submission of interaction data
mandatory<br>
        </li>
        <li>Coordination of curation.</li>
      </ul>
    </ul>
    <li>
      <h3><font color="#000000"><a name="miape-detailed-agenda"

 id="miape-detailed-agenda"></a></font>MIAPE Development -- Integrating Proteomics 
        Workflows (Chris Taylor)<font color="#000000"><strong></strong></font></h3>
      <h3><font color="#000000"><strong></strong></font></h3>
      <br>
      <ul>
        <li><strong><em>Tuesday 20th (morning):</em></strong><em> Short talks 
          series</em><br>
          <ul>
            <li>ProteomeGRID &#8211; Distributed computing for 2DE image analysis. 
              <em>Andrew Dowsey, Imperial College, London, UK.</em></li>
            <li>The Sashimi tool suite and mzXML data format. <em>Patrick Pedrioli, 
              ISB, Seattle WA, USA.</em></li>
            <li>The Proteios open source proteomics initiative. <em>Jari H&auml;kkinen, 
              Lund University , Sweden &amp; Frank Potthast, FGC Zurich, Switzerland.<br>
              <font color="#990000"><strong>-- Coffee break -- </strong></font></em></li>
            <li>The development of the MAGE model. <em>Ugis Sarkans, EBI, Hinxton, 
              UK.</em></li>
            <li>Adapting MAGE for specific communities. <em>Philippe Rocca-Serra, 
              EBI, Hinxton, UK.</em></li>
            <li>CEBS object model for systems biology data, SysBio-OM. <em>Sandhya 
              Xirasagar, NIEHS NCT (CEBS), Research Triangle Park NC, USA.</em></li>
            <li>FGE-OM: Functional Genomics Experiment Object Model. <em>Andrew 
              Jones, University of Glasgow, UK.<br>
              <br>
              </em></li>
          </ul>
        </li>
        <li><strong><em>Tuesday 20th (afternoon):</em></strong><em> We go to work...</em><br>
          <ul>
            <li>The MIAPE 'Principles' document.</li>
            <li>A walk through MIAPE 0.1<br>
              <em><font color="#990000"><strong>-- Teatime --</strong></font></em></li>
            <li>MIAPE model development session<br>
            </li>
          </ul>
        </li>
      </ul>
    </li>
  </ul>
</ul>
<p>&nbsp;</p>
<h3><a name="logistics"></a>Logistics</h3>
The meeting will take place in the Hotel Kyriad, Carros, near Nice,
Cote d'Azur, France.<a
 href="http://www.hotel-kyriad-carros.cote.azur.fr/"><img
 src="http://www.hotel-kyriad-carros.cote.azur.fr/images/photo1_3.jpg"
 title="" alt=""
 style="border: 0px solid ; width: 200px; height: 150px;" align="right"></a><br>
<a href="http://www.hotel-kyriad-carros.cote.azur.fr/page_en_1.html">http://www.hotel-kyriad-carros.cote.azur.fr/page_en_1.html</a><br>
<br>
Registration for the meeting is now closed.<br>
<h4>Access</h4>
<pre>Hotel Kyriad<br>Carros<br>1ere Avenue<br>06510 Carros<br>France<br>Tel +33 493 08 77 80<br>Fax +33 493 08 73 96<br></pre>
Taxi: Approximately 40 Euro from Nice airport.<br>
Public transport: Not much, but one very nice, though time-consuming
option:<br>
<ul>
  <li>Bus 98 from the airport to "Nice Gare SNCF". Attention, you have
to tell the driver you want to go there!</li>
  <li>Walk across to "Gare Chemins de fer de Provence" according to the
map<br>
    <a href="http://www.trainprovence.com/Ftp/1512-I.jpg">http://www.trainprovence.com/Ftp/1512-I.jpg</a></li>
  <li>Take the picturesque "Train des Pignes":<br>
http://www.trainprovence.com</li>
  <li>The stop is "Colomars la Manda", the 500 metres walk to the
hotel. <br>
  </li>
</ul>
<p>Map: <a
 href="http://www.multimap.com/map/browse.cgi?X=801250&amp;Y=5400000&amp;width=700&amp;height=400&amp;client=public&amp;gride=&amp;gridn=&amp;srec=0&amp;coordsys=mercator&amp;addr1=&amp;addr2=1ere+avenue&amp;addr3=carros&amp;pc=06510&amp;advanced=&amp;local=&amp;out.x=7&amp;out.y=3&amp;scale=50000">http://www.multimap.com/map/browse.cgi?X=801250&amp;Y=5400000&amp;width=700&amp;height=400&amp;client=public&amp;gride=&amp;gridn=&amp;srec=0&amp;coordsys=mercator&amp;addr1=&amp;addr2=1ere+avenue&amp;addr3=carros&amp;pc=06510&amp;advanced=&amp;local=&amp;out.x=7&amp;out.y=3&amp;scale=50000</a>
</p>
<hr style="width: 100%; height: 2px;">
<h3 style="text-align: center;"><a name="acknowledgements"
 class="mozTocH3"></a></h3>
<h3 style="text-align: center;">Sponsored by</h3>
<br>
<table cellpadding="2" cellspacing="2" border="0"
 style="text-align: left; width: 100%;">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: center;">&nbsp; <img
 src="http://newsroom.lilly.com/images/corpsite/logo.gif" title=""
 alt="Lilly" style="width: 94px; height: 69px;"><br>
      </td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/novartis.jpg" title="" alt="Novartis"
 style="width: 214px; height: 45px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/roche.png" title="" alt="Roche"
 style="width: 90px; height: 49px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/serono.png" title="" alt="Serono"
 style="width: 153px; height: 47px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/syngenta.jpg" title="" alt="Syngenta"
 style="width: 150px; height: 64px;"></td>
    </tr>
  </tbody>
</table>
<table cellpadding="2" cellspacing="2" border="0"
 style="text-align: left; width: 100%;">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: center;"><br>
      </td>
      <td style="vertical-align: top; text-align: center;"><br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: center;"><img
 src="http://www.ebi.ac.uk/industry/images/logo_industry.gif" title=""
 alt="Industry logo" style="width: 113px; height: 75px;" align="top"><br>
      </td>
      <td style="vertical-align: top; text-align: center;">
      <h3><a href="http://www.hupo.org/"><img
 alt="HUPO Human Proteome Organisation" src="../../images/hupo.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="middle"
 title=""></a></h3>
      </td>
    </tr>
  </tbody>
</table>
<br>
<hr>
<address>Henning Hermjakob, hhe@ebi.ac.uk, 2003-Nov-27, 2004-Apr-15<br>
</address>
<!-- Created: Mon Oct  7 13:24:01 BST 2002 --><!-- hhmts start --><!-- hhmts end -->
</body>
</html>
