<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>PSI spring workshop 2004</title>
</head>
<body>
<h3 style="text-align: center;"><a name="mozTocId830364"
 class="mozTocH3"></a><a href="http://psidev.sourceforge.net"><img
 alt="Proteomics Standards Initiative" src="../../images/psi.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="left"
 title=""></a> <a href="http://www.hupo.org/"><img
 alt="HUPO Human Proteome Organisation" src="../../images/hupo.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="right"
 title=""></a>HUPO Proteomics Standards Initiative <br>
</h3>
<h3 style="text-align: center;"><a name="mozTocId313157"
 class="mozTocH3"></a>Spring Workshop <br>
</h3>
<h3 style="text-align: center;"><a name="mozTocId689138"
 class="mozTocH3"></a>April 17-21, 2004</h3>
<h3 style="text-align: center;"><a name="mozTocId528852"
 class="mozTocH3"></a>Carros, near Nice, Cote d'Azur, France</h3>
<h3 style="text-align: center;">Meeting minutes</h3>
<br>
<b><span style="font-family: Arial;"><o:p></o:p></span></b>
<p class="MsoBodyText" align="left" style="text-align: left;">The
Spring 2004 meeting
of the PSI was divided into two halves. For the first two days, members
who had
been actively involved in the development of the standards over the
previous 15
months were invited to work sessions to further develop these schemas.
The
second half of the meeting was thrown open to all interested parties
who were
interested in becoming involved in the future development of the
various models
and tools or in implementing them into their in house workflows.</p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p></o:p></span></p>
<h2><b><span style="font-family: Arial;">Session 1<o:p></o:p></span></b></h2>
<p class="MsoNormal"><b><span style="font-family: Arial;">Introduction</span></b><span
 style="font-family: Arial;"><span style="">&nbsp;&nbsp; </span>Delegates
were welcomed to the opening session by Henning Hermjakob (EBI) who
briefly
summarised the achievements of the PSI since its inception in 2002. The
Molecular Interactions group published the PSI-MI Level 1 XML schema at
the
beginning of 2004 and most of the major molecular interaction databases
are
already offering data in that format. The Mass Spectrometry group have
presented their preliminary PSI-MS mzData interchange format at the
HUPO 2003
congress in </span><st1:city><st1:place><span
 style="font-family: Arial;">Montreal</span></st1:place></st1:city><span
 style="font-family: Arial;"> and had since been actively involved in
discussion
in how to refine this schema prior to publication. Finally, work has
started on
developing the MIAPE data model which, it is envisaged, will enable
users to
describe all the relevant data pertaining to a proteomics experiment in
a
standardised and clearly understood format. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp; <br>
</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">The sessions
then divided
into the 3 work groups<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">Molecular
Interactions<span style="">&nbsp; </span>- session chair Henning
Hermjakob (EBI)<o:p></o:p></span></b></p>
<p class="MsoNormal"><span style="font-family: Arial;">The session
started with a
review of the issues raised by the current implementation of the
schema. <o:p></o:p></span></p>
<ul style="margin-top: 0cm;" type="disc">
  <li class="MsoNormal" style=""><span style="font-family: Arial;">IDs
to revert to text &#8211; XML requires this to be a unique feature otherwise
a problem is caused as the file collapses from the extended to the
compact format. XML ID type can be changed to a string.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">ID
attribute required for Interactions &#8211; this will ease indexing.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">ParticipantDetection
rename as ParticipantIdentification<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Method
added to all appropriate fields e.g. ParticipantIdentificationMethod &#8211;
this will break the existing code.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Complex
topology &#8211; the description of this was seen as a major issue which was
held over for later discussion<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Feature
identification and linking &#8211; at this point it was resolved to solve
this with the assigning of Feature ID attributes<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Feature
location is currently mandatory; this should become optional, with
option of &#8220;Undetermined location&#8221; added. The difference between Site
and Position was not clearly documented and needed to be resolved.
Discontinuous features should be dealt with as a recursive list &#8211; the
EMBL/DDBJ/GenBank approach to this could be adopted. <o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Protein
feature to become a free text description. FeatureDescription to be
renamed as FeatureType, current XML type will require renaming.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">FeatureNames
to be rechristened Names and made a free text description. Type should
be extended to allow optional addition of aliases.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Confidence
should be made a Type to allow assignment of more than 1 value. All
databases agreed this should be used to reproduce author-assigned
confidence values rather than any assignment by database curators.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Inconsistency
in ordering of cross references and bibliographic references resolved.
A bibliographic reference should be mandatory with documentation on how
to deal with direct submissions prior to publication. The nucleotide
database policy can be adopted.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Minimum
number of ProteinParticipants needs to be reduced to 1 to allow for the
description of intramolecular interactions.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">All
protein interactors should contain either a UniProt accession or GI
number to be placed in cross references. The ability to describe
sequence should be allowed but users should be encouraged to deposit to
UniProt and acquire an accession number. Proteins will be identified
within an entry by a unique identifier, any merging process will have
to resolve potential conflicts. Databases should document process for
assigning protein IDs to large datasets &#8211; particularly partial genes
which may be one of several isoforms.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Controlled
vocabulary for databases established &#8211; BIND to supply initial
definitions.<o:p></o:p></span></li>
  <li class="MsoNormal" style=""><span style="font-family: Arial;">Controlled
vocabulary editorial board established, a voting process over 5 days
(default will be positive) agreed upon.<o:p></o:p></span></li>
</ul>
<p class="MsoNormal" style="margin-left: 36pt;"><span
 style="font-family: Arial;">Sandra
Orchard &#8211; IntAct (Co-ordinator)<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt;"><span lang="IT"
 style="font-family: Arial;">Luisa Montecchi &#8211; MINT<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt;"><span lang="IT"
 style="font-family: Arial;">Susan Moore &#8211; BIND<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt;"><span
 style="font-family: Arial;">Lukasz
Salwinski </span><span lang="IT" style="font-family: Arial;">&#8211;
DIP</span><span style="font-family: Arial;"><o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt;"><span
 style="font-family: Arial;">John
Armstrong &#8211; GSK<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 18pt; text-indent: -18pt;"><!--[if !supportLists]--><span
 style="font-family: Symbol;"><span style="">&middot;<span
 style="font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant: normal; font-weight: normal; font-size: 7pt; line-height: normal; font-stretch: normal; font-size-adjust: none;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><!--[endif]--><span style="font-family: Arial;">Exporting
databases should label curated entries with their database ID in the
cross
reference line, importing databases should retain label on import.<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 18pt; text-indent: -18pt;"><!--[if !supportLists]--><span
 style="font-family: Symbol;"><span style="">&middot;<span
 style="font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant: normal; font-weight: normal; font-size: 7pt; line-height: normal; font-stretch: normal; font-size-adjust: none;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><!--[endif]--><span style="font-family: Arial;">Additional
attribute to be added to cross reference field to store evidence
codes/qualifiers.<o:p></o:p></span></p>
<p class="MsoNormal"><!--[if !supportLists]--><span
 style="font-family: Symbol;"><span style="">&middot;<span
 style="font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant: normal; font-weight: normal; font-size: 7pt; line-height: normal; font-stretch: normal; font-size-adjust: none;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><!--[endif]--><span style="font-family: Arial;">GO
to be written
as GO:0000XXX, MI as MI:0000XXX<o:p><br>
<br>
</o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">Mass
Spectrometry</span></b><span style="font-family: Arial;"> &#8211; <b>session
chairs Randall Julian (Eli Lilly) and
Weimin Zhu (EBI)<br>
</b></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><b><o:p></o:p></b></span></p>
<p class="MsoNormal"><a href="msOpening.ppt">(Opening presentation)</a><br>
</p>
<p class="MsoNormal"><span style="font-family: Arial;">The main aim of
the initial
mass spectrometry session was to demonstrate the mzData format to the
vendors
and take in their feedback. The first decision taken was to extend the
acquisition field to encompass both mass array and intensity array,
both of
which would become mandatory fields. The response from both
instrumentation and
search engine manufacturers was generally good, but tempered with a
warning
that an exponential rise could be expected in the volume of data being
generated by the next generation of high throughput machines which
would
compromise the ability of XML to produce files of a practical volume.
Future
coding will be in Base64 to produce a more compact file size. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Controlled
vocabularies need
to be developed for parameters such as the source, detection method and
analysis with the ability for the development of user controlled
vocabularies
to allow for the specific needs of laboratories and diversity in
experimental
methodology. It was proposed that the existing ASMS vocabulary list be
looked
at in detail in the full session, with a view to updating this and
adopting
this as the data source for the controlled vocabulary. Finally, the
development
of a common output format for MS data analysis was discussed, with
common
syntax being developed to describe peptide and protein identification
and for
detailing post-translational modifications.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">MzData is
designed to allow
data sharing and access but will also be an element of the MIAPE model
for
describing a proteomics model.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">MIAPE</span></b><span
 style="font-family: Arial;"> <b>&#8211; session chair Chris Taylor (EBI)<o:p></o:p></b></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">MIAPE is being
designed as a
robust, future-proofed standard representation of both methods and data
from
proteomics experiments. It will be a modular expansible model of
primary data
in a format which will allow search, analysis, tool development and
information
exchange. The MIAPE modelling group propose to produce two documents<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 54pt; text-indent: -18pt;"><!--[if !supportLists]--><span
 style="font-family: Arial;"><span style="">1.<span
 style="font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant: normal; font-weight: normal; font-size: 7pt; line-height: normal; font-stretch: normal; font-size-adjust: none;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><!--[endif]--><span style="font-family: Arial;">MAIPE
Principles which will describe the scope and
the purpose of the proteomics model<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 54pt; text-indent: -18pt;"><!--[if !supportLists]--><span
 style="font-family: Arial;"><span style="">2.<span
 style="font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant: normal; font-weight: normal; font-size: 7pt; line-height: normal; font-stretch: normal; font-size-adjust: none;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><!--[endif]--><span style="font-family: Arial;">MAIPE
model &#8211; a documented UML model which will map
to an XML schema, with explicit documentation on the capture of both
data and
meta-data and reference implementation.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">It is envisaged
that this
model will develop further following publication on the Web site via
the e-mail
list, teleconferences and further meetings.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">The Principles
should
include the assessment of appropriate experimentation and design and
allow the
results of different experiments to be compared with the evidence that
was used
to obtain derived data being made clear.<span style="">&nbsp;
</span>The model will be designed to underpin public data repositories
of
proteomics data both to support the sharing of data between projects
where
different aspects of experimental activities, including reanalysis of
data, is
taking place at multiple centres and to support the description of
composite
data sets to allow benchmarking and comparison. It will also support
the
development of general analytical tools, which can then be made
available to
the proteomics community as a whole.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<h2>Session 2</h2>
<p class="MsoNormal"><span style="font-family: Arial;">Delegates at the
open
session were again welcomed by Henning Hermjakob and the session chairs
each
gave a brief overview of the progress made to date and their aims for
the
remainder of the meeting. A presentation on the development of the MGED
model
was then given to the entire meeting, as an example of the analogous
process of
standardisation in the field of microarray technologies. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<h3>The MGED Standardisation process: Angel Pizarro</h3>
<p class="MsoNormal"><span style="font-family: Arial;">The MGED
consortium was
formed in 1999 and published MIAME, an informal specification/check
list to be
followed when publishing a microarray experiment in 2001 followed by
MAGE-OM in
2002 which is a formalisation of concepts in MIAME into an
implementation
phase. The MGED Ontology (MO) was derived from the MIAME guidelines and
has
been published in DAML and OIL files to provide standard annotation for
the
annotation of microarray experiments. This is the major document for
data entry
into MGED. As well as allowing the representation of simple concepts,
the
current data model also allows the modelling of more complex
situations. Use
cases drove the subsequent development of formal documented procedures.
Version
2.0 will be published in the near future and will be part of a more
inclusive functional
genomics model, with data specifications encoded within the model
itself. A
common set of support models could be shared with MIAPE e.g. Protocol,
BioMaterial, BioAssay and Audit, providing a roadmap for MIAPE
development and
eventually suggesting that complex experiments encompassing both
genomic and
proteomic data could be described within a single data model.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<h2>Molecular Interactions</h2>
<span style="font-weight: normal;">The second session was opened by a
series
of short presentations on the implantation of PSI-MI by the
participating
databases.<o:p></o:p></span>
<br>
<o:p>&nbsp;</o:p>
<p class="MsoNormal"><b><span style="font-family: Arial;">IntAct &#8211;
Henning
Hermjakob (EBI)</span></b><span style="font-family: Arial;"><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">IntAct was
initiated in 2002
by 8 partners, co-ordinated by the EBI and funded by the EU as part of
the Framework
5 &#8220;Temblor&#8221; grant. PSI-MI is the native download format of IntAct with
large
datasets such as the Giot drosophila 20,000 interactions being split
into 2.500
interaction portions. Data can also be searched and downloaded using
Graph2MIF
and it is possible to download local interaction graphs using PSI-XML
format.
Import using PSI-MI is also in place and data exchange with other
databases is
already established (MINT) or planned in the near future (DIP).<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span><span
 style="font-family: Arial;"><a href="intact.ppt">(Presentation)</a></span><span
 style="font-family: Arial;"><o:p></o:p></span></p>
<h3>MIPS &#8211; Volker Stuempflen</h3>
<p class="MsoNormal"><span style="font-family: Arial;">MIPS provides
bioinformatics
support for genome analysis. The interaction database contains manually
curated
mammalian data and both manually entered and large scale yeast data. In
order
to accommodate PSI-MI, the current table schema is being revised with
the
eventual aim of MIPS being able to derive PSI-MI files. This fits in
well with
the HOBIT inter-database collaboration currently established in </span><st1:country-region><st1:place><span
 style="font-family: Arial;">Germany</span></st1:place></st1:country-region><span
 style="font-family: Arial;"> in which data will be made available
using WSDL (Web
services descriptive language) as the standard definition for data and
methods
but implementation is the responsibility of the consumer or service
provider.
Internal restructuring to PSI-MI is now complete but a style sheet
needs to be
written to complete the web interface.<span style="">&nbsp; </span><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p><a
 href="MIPS_PSIMI.ppt">(Presentation)</a><br>
</o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">BioPax - Gary
Bader
(Sloan Kettering Canner Centre)<o:p></o:p></span></b></p>
<p class="MsoNormal"><span style="font-family: Arial;">BioPax is a data
exchange
ontology for biological pathways which currently maintains a curated
list of
104 pathway related resources. An entire pathway is contained in a
single
BioPax record using existing standards, including PSI-MI with which it
is
intended to remain totally compatible. Different preferred
representations can
then be described using BioPax. An ontology has been developed for the
project
using OWL/XML schema. Version 0.5 was released in September 2004 and a
levelled
approached is planned &#8211; level 1 will allow the representation of
biochemical
reactions, complex assembly and enzyme and transport catalysis, level 2
will
add binding interactions using PSI-MI and level 3 will bring in genetic
interactions and regulation. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Cytoscape allows
visualisation of the biomolecular networks and is an open-source
Java-based
tool with plug-ins for network analysis. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span><a
 href="PSI%20Apr_2004%20-%20bader.ppt">(Presentation)</a><span
 style="font-family: Arial;"><o:p></o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">MINT - Luisa
Montecchi (</span></b><st1:place><st1:placetype><b><span
 style="font-family: Arial;">University</span></b></st1:placetype><b><span
 style="font-family: Arial;"> of </span></b><st1:placename><b><span
 style="font-family: Arial;">Rome</span></b></st1:placename></st1:place><b><span
 style="font-family: Arial;">)<o:p></o:p></span></b></p>
<p class="MsoNormal"><span style="font-family: Arial;">MINT has been
restructured
to the basic PSI-structure and controlled vocabularies have been
implemented.
Since MINT uses flat file based input/output tools in the curation
process, a
PSI maker/flattener has been developed which uses a dictionary as a
filter to
remove any data not appropriate for PSI.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span><a
 href="mint_psi_nice.ppt">(Presentation)</a><span
 style="font-family: Arial;"><o:p></o:p></span></p>
<h3>TransPath</h3>
<span style="font-family: Arial;">TransPath exists as a
poly-hierarchy with a chain or pathway defined by a set of reactions
joined by
common molecules. A pathway step defines a biochemical change supported
by
molecular evidence with an assigned quality value. Several reactions
can be
grouped to one chain and several chains to one pathway. TransPath will
become
public to academics for web browsing in May 2004. <o:p></o:p></span>
<o:p>&nbsp;<br>
<br>
</o:p><a href="transpath_psi2004.ppt">(Presentation)</a> <br>
<h3>BIND &#8211; John Salama</h3>
<span style="font-weight: normal;">BIND describes interactions,
complexes and
pathways involving proteins, nucleic acids, genes and small molecules.
Version
4.0 will include support for PSI import. Java-based submission and
curation
software are freely available.<o:p></o:p></span>
<br>
<span style="">&nbsp;</span>
<p class="MsoNormal"><b><span style="font-family: Arial;">Schema Updates</span></b><span
 style="font-family: Arial;"> <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Following a
short
discussion, it was decided that the schema would undergo an annual
update cycle<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Month 0:<span
 style="">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Change to new schema
on master copy<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Month &#8211;2:<span
 style="">&nbsp;&nbsp;&nbsp; </span>Freeze of development schema and
announcement via usual <o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt; text-indent: 36pt;"><span
 style="font-family: Arial;">media<span style="">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span><o:p></o:p></span></p>
<p class="MsoNormal" style=""><span style="font-family: Arial;"><span
 style="">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>Testing<o:p></o:p></span></p>
<p class="MsoNormal" style=""><span style="font-family: Arial;"><span
 style="">&nbsp;&nbsp;&nbsp; </span><span style="">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>Implementation<o:p></o:p></span></p>
<p class="MsoNormal" style=""><span style="font-family: Arial;">Month
&#8211;4.5: Production of new development schema<span style="">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Month &#8211;5:<span
 style="">&nbsp;&nbsp;&nbsp; </span>Developers meeting<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Month &#8211;5.5:
Request changes
frozen<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">Should a vote be
required at
the developers meeting each organisation currently participating would
be allowed
1 vote, namely IntAct, BIND, DIP, MINT, MIPS, BioPax and, representing
industry, Hybrigenics, Glaxo and Serono with a simple majority
required. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<h3>Schema Extensions</h3>
<ol style="margin-top: 0cm;" start="1" type="1">
  <li class="MsoNormal" style=""><b><span style="font-family: Arial;">Negative
information </span></b><span style="font-family: Arial;">&#8211; will be
included but with a mandatory flag at the interaction level. The
default within the schema will be positive. Curation standards will
need to be agreed upon and all such data will be separately maintained
on exchange so that importing databases can choose not to take it.<o:p></o:p></span></li>
</ol>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<ol style="margin-top: 0cm;" start="2" type="1">
  <li class="MsoNormal" style=""><span style="font-family: Arial;">New
interactor types &#8211; DNA, RNA and small molecules. All will be new
Interactor Types with an internal identifier. A free text feature space
will be retained for the nucleic acids.<o:p></o:p></span></li>
</ol>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<ol style="margin-top: 0cm;" start="2" type="1">
  <ol style="margin-top: 0cm;" start="1" type="a">
    <li class="MsoNormal" style=""><span style="font-family: Arial;">Small
molecules &#8211; nomenclature will be an issue. A database list will need to
be added, an attribute list with further specification. Allowed
databases &#8211; ChEBI, Kegg<o:p></o:p></span></li>
    <li class="MsoNormal" style=""><span style="font-family: Arial;">DNA
&#8211; binding positions will need to be described which will require the
ability to deal with start site changes. Absolute positions are more
prone to change than relative. Nucleotide databases to be consulted on
issue. Need to extend location type to relative (upstream of (-),
downstream of (+)). Nucleotide AC nos plus version will need to be
given. Also need ability to describe &#8220;fuzzy&#8221; locations and Sequence
will remain an option but with a strong recommendation that this will
not be used.<o:p></o:p></span></li>
    <li class="MsoNormal" style=""><span style="font-family: Arial;">RNA
&#8211; acceptable AC nos. EMBL/DDBJ/Genbank, Ensembl, Entrez-Gene and GI
numbers.<o:p></o:p></span></li>
  </ol>
</ol>
<p class="MsoNormal"><b><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></b></p>
<p class="MsoNormal" style=""><b><span
 style="font-size: 10pt; font-family: &quot;Courier New&quot;;"><o:p>&nbsp;</o:p></span></b></p>
<p class="MsoNormal" style="margin-left: 36pt; text-indent: -18pt;"><b><span
 style="font-family: Arial;">3.<span style="">&nbsp;&nbsp; </span></span></b><span
 style="font-family: Arial;">Complex
Topology &#8211; current schema does not allow this to be accurately
represented. </span><span style="font-family: Arial;">A separate
inferredInteraction field was added inside interaction that handles
topology
and many experiments backing up one interaction, since both of these
are
similar (in that they are both inferred).</span><span
 style="font-size: 10pt; font-family: &quot;Courier New&quot;;"><o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal" style="margin-left: 36pt; text-indent: -18pt;"><b><span
 style="font-family: Arial;">4.</span></b><span
 style="font-family: Arial;"><span style="">&nbsp; </span>n </span><span
 style="font-family: Symbol;"><span style="">:</span></span><span
 style="font-family: Arial;"> m interactions &#8211; although this was
originally
allowed for, the existing schema will not allow BIND to import the
information
required to fully describe an interaction under their database
definition. </span><span style="font-family: Arial;">The specification
will be
adjusted to solve the issues surrounding specifying experimental form
of
participants (e.g. isTagged) for each by separating the experimental
form items
from the participant and moving them to a separate experimental form
section
within the interaction.<span style="">&nbsp; </span>Each
experimental form will then point to the experiment it is part of.<o:p></o:p></span></p>
<p class="MsoNormal" style="margin-left: 18pt;"><span
 style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<ol style="margin-top: 0cm;" start="5" type="1">
  <li class="MsoNormal" style=""><span style="font-family: Arial;">A
large number of updates and additions to the controlled vocabularies
were agreed upon by the editorial board.<o:p></o:p></span></li>
</ol>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">The current
agreed changes
are extensive enough to require upgrade from Version 1.0 to 2.0 &#8211;
projected
release date </span><st1:date year="2004" day="15" month="9"><span
 style="font-family: Arial;">15<sup>th</sup> September 2004</span></st1:date><span
 style="font-family: Arial;">. It was agreed this would be published by
submitting
a paper to BMC Bioinformatics, with all attendees plus PIs as authors.
The new
version would be accompanied by appropriate examples and use cases &#8211;
David
Sherman to co-ordinate.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><b><span style="font-family: Arial;">Query System </span></b><span
 style="font-family: Arial;">&#8211; A common web-based query face was
discussed in
which search on a protein AC number will give protein plus interators
(to a
depth of 2/3). Initial technical specifications to be written by MIPS. <o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span><a
 href="miConclusion.ppt">(Short
summary of molecular interactions session)</a><span
 style="font-family: Arial;"><o:p></o:p></span></p>
<h2>Mass Spectrometry</h2>
<p class="MsoNormal"><o:p>&nbsp;</o:p></p>
<p class="MsoNormal"><span style="font-family: Arial;">The Mass
Spectrometry group
moved on to consider the extension of the controlled vocabulary list,
currently
provided by the ASMS. It was agreed that this needed both adding to and
updating to cover current requirement and all attendees were asked to
contribute to this process. It was agreed that the ASMS would be an
appropriate
expert group to consult on this process and that eventual ownership of
this
list could move to the American Society for Mass Spectrometry (ASTM) to
support
both PSI-mzData and their own raw data standardisation efforts.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">mzData<span
 style="">&nbsp; </span>is now widely regarded as an acceptable
format for representing mass spectrometry data and will be valuable for
both
data sharing and storage. Once the modifications agreed upon by meeting
are in
place, the schema will be frozen prior to implementation and
publication. Those
vendors who have agreed to be involved at an early stage of the
process, for
example Matrix Science, </span><span style="font-family: Arial;">Kratos
Analytical Ltd</span><span style="font-family: Arial;">
and Bruker Daltronics will experiment with the format to establish
proof-of-concept for spectral and parameter storage. It is also
intended that a
public repository for such data will be available before the HUPO
congress in
October 2004. In the long term, however, it was accepted that peak
lists are
too limiting for most reprocessing and that any archive will be
required to
hold raw data. To that end, it is recommended that the ASTM standard be
used
for full data archiving, when available (2005-2006).<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;">It was agreed
that sample
description, separation and identification should be kept separate.<span
 style="">&nbsp; </span>The mzData model will also act as the mass
spectrometry component of the MIAPE data model and alignment with this
must be
kept in mind throughout the development process, for example top line
processes
such as sample identification will be better dealt with by MIAPE than
in
isolation by the mass spectrometry group. The design of specifications
for a
spectral analysis format (mzAnalysis) is now in hand. This format will
require
the development of a common syntax for the identification of proteins
and
peptides and also the ability to describe post-translational
modifications
(mzProtID). Any such format must facilitate both database searching and
the
ability to annotate denovo sequencing and also quantitation of
chromatographic
and spectrometric data. Vendor dependence will be managed through the
use of
ontologies.<o:p></o:p></span></p>
<p class="MsoNormal"><span style="font-family: Arial;"><o:p>&nbsp;</o:p></span></p>
<h3><b><span style="font-family: Arial;">General Proteomics Schema/MIAPE<o:p></o:p></span></b></h3>
<h3><b><span style="font-family: Arial;"><o:p></o:p></span></b></h3>
<span style="font-family: arial;">Please see the detailed report at <a
 href="http://psidev.sourceforge.net/gps/carros/index.html">http://psidev.sourceforge.net/gps/carros/index.html</a><br>
<br>
<br>
</span>
<hr style="width: 100%; height: 2px;">
<h3 style="text-align: center;"><a name="acknowledgements"
 class="mozTocH3"></a></h3>
<h3 style="text-align: center;">The meeting was generously sponsored by</h3>
<br>
<table cellpadding="2" cellspacing="2" border="0"
 style="text-align: left; width: 100%;">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: center;">&nbsp; <img
 src="http://www.lilly.com/images/lilly_globalNav_sm2.gif" title=""
 alt="Lilly" style="width: 94px; height: 69px;"><br>
      </td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/novartis.jpg" title="" alt="Novartis"
 style="width: 214px; height: 45px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/roche.png" title="" alt="Roche"
 style="width: 90px; height: 49px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/serono.png" title="" alt="Serono"
 style="width: 153px; height: 47px;"></td>
      <td style="vertical-align: top; text-align: center;"><img
 src="../../images/syngenta.jpg" title="" alt="Syngenta"
 style="width: 150px; height: 64px;"></td>
    </tr>
  </tbody>
</table>
<table cellpadding="2" cellspacing="2" border="0"
 style="text-align: left; width: 100%;">
  <tbody>
    <tr>
      <td style="vertical-align: top; text-align: center;"><br>
      </td>
      <td style="vertical-align: top; text-align: center;"><br>
      </td>
    </tr>
    <tr>
      <td style="vertical-align: top; text-align: center;"><img
 src="http://www.ebi.ac.uk/industry/images/logo_industry.gif" title=""
 alt="Industry logo" style="width: 113px; height: 75px;" align="top"><br>
      </td>
      <td style="vertical-align: top; text-align: center;">
      <h3><a href="http://www.hupo.org/"><img
 alt="HUPO Human Proteome Organisation" src="../../images/hupo.gif"
 style="border: 0px solid ; width: 113px; height: 75px;" align="middle"
 title=""></a></h3>
      </td>
    </tr>
  </tbody>
</table>
<br>
<hr>
<address>Henning Hermjakob, hhe@ebi.ac.uk, 2004-July-15<br>
</address>
<!-- Created: Mon Oct  7 13:24:01 BST 2002 --><!-- hhmts start --><!-- hhmts end -->
</body>
</html>
