<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="Content-Type"
 content="text/html; charset=windows-1252">
  <meta name="Generator" content="Microsoft Word 10 (filtered)">
  <title>Meeting of the HUPO-PSI group</title>
  <style id="dynCom" type="text/css"><!-- --></style>
  <script language="JavaScript"><!--
function msoCommentShow(anchor_id, com_id)
{
	if(msoBrowserCheck()) 
		{
		c = document.all(com_id);
		a = document.all(anchor_id);
		if (null != c && null == c.length && null != a && null == a.length)
			{
			var cw = c.offsetWidth;
			var ch = c.offsetHeight;
			var aw = a.offsetWidth;
			var ah = a.offsetHeight;
			var x  = a.offsetLeft;
			var y  = a.offsetTop;
			var el = a;
			while (el.tagName != "BODY") 
				{
				el = el.offsetParent;
				x = x + el.offsetLeft;
				y = y + el.offsetTop;
				}
			var bw = document.body.clientWidth;
			var bh = document.body.clientHeight;
			var bsl = document.body.scrollLeft;
			var bst = document.body.scrollTop;
			if (x + cw + ah / 2 > bw + bsl && x + aw - ah / 2 - cw >= bsl ) 
				{ c.style.left = x + aw - ah / 2 - cw; }
			else 
				{ c.style.left = x + ah / 2; }
			if (y + ch + ah / 2 > bh + bst && y + ah / 2 - ch >= bst ) 
				{ c.style.top = y + ah / 2 - ch; }
			else 
				{ c.style.top = y + ah / 2; }
			c.style.visibility = "visible";
}	}	}
function msoCommentHide(com_id) 
{
	if(msoBrowserCheck())
		{
		c = document.all(com_id);
		if (null != c && null == c.length)
		{
		c.style.visibility = "hidden";
		c.style.left = -1000;
		c.style.top = -1000;
		} } 
}
function msoBrowserCheck()
{
	ms = navigator.appVersion.indexOf("MSIE");
	vers = navigator.appVersion.substring(ms + 5, ms + 6);
	ie4 = (ms > 0) && (parseInt(vers) >= 4);
	return ie4;
}
if (msoBrowserCheck())
{
	document.styleSheets.dynCom.addRule(".msocomanchor","background: infobackground");
	document.styleSheets.dynCom.addRule(".msocomoff","display: none");
	document.styleSheets.dynCom.addRule(".msocomtxt","visibility: hidden");
	document.styleSheets.dynCom.addRule(".msocomtxt","position: absolute");
	document.styleSheets.dynCom.addRule(".msocomtxt","top: -1000");
	document.styleSheets.dynCom.addRule(".msocomtxt","left: -1000");
	document.styleSheets.dynCom.addRule(".msocomtxt","width: 33%");
	document.styleSheets.dynCom.addRule(".msocomtxt","background: infobackground");
	document.styleSheets.dynCom.addRule(".msocomtxt","color: infotext");
	document.styleSheets.dynCom.addRule(".msocomtxt","border-top: 1pt solid threedlightshadow");
	document.styleSheets.dynCom.addRule(".msocomtxt","border-right: 2pt solid threedshadow");
	document.styleSheets.dynCom.addRule(".msocomtxt","border-bottom: 2pt solid threedshadow");
	document.styleSheets.dynCom.addRule(".msocomtxt","border-left: 1pt solid threedlightshadow");
	document.styleSheets.dynCom.addRule(".msocomtxt","padding: 3pt 3pt 3pt 3pt");
	document.styleSheets.dynCom.addRule(".msocomtxt","z-index: 100");
}
// --></script>
  <style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:Arial;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:Arial;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:Arial;
	font-weight:bold;}
ins
	{text-decoration:none;}
span.msoIns
	{text-decoration:underline;}
span.msoDel
	{text-decoration:line-through;
	color:red;}
@page Section1
	{size:595.3pt 841.9pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
  </style>
</head>
<body lang="EN-GB" link="blue" vlink="purple">
<div class="Section1">
<div style="text-align: left;"></div>
<p class="MsoNormal" style="text-align: right;"><b><img
 src="http://psidev.sourceforge.net/images/psi.gif" title=""
 alt="PSI logo" style="width: 113px; height: 75px;" align="left"><img
 src="http://psidev.sourceforge.net/images/hupo.gif" title=""
 alt="HUPO logo" style="width: 113px; height: 75px;"></b></p>
<p class="MsoNormal" align="center" style="text-align: center;"><b>Meeting
Report
from the</b></p>
<p class="MsoNormal" align="center" style="text-align: center;"><b>2nd
Annual HUPO
congress, </b><b>Montreal</b></p>
<p class="MsoNormal" align="center" style="text-align: center;"><b>9<sup>th</sup>
October 2003</b></p>
<p class="MsoNormal" align="center" style="text-align: center;"><b>&nbsp;</b></p>
<p class="MsoNormal">The need for common standards which will allow the
interchange, comparison and public domain deposition of published
proteomic
data is a recognised need within the scientific community. To this end,
the
Proteomic Standards Initiative (PSI) of the Human Proteomic
organisation (HUPO)
has been working to develop such standards in collaboration with major
database
providers, instrumentation and software providers and other
standardisation
efforts in related fields. The PSI contribution to the second annual
HUPO
congress was divided into two parts. A full session at the conference,
chaired
by Rolf Apweiler and Weimin Zhu, was dedicated to describing the
progress made
to date in developing standards and interchange formats in the fields
of
protein-protein interactions (PPI) and mass spectrometry, whilst also
highlighting areas where such efforts have yet to be initiated, for
example in
the storage of information from 2-D electophoresis gels. Secondly, the
PSI held
an open meeting at which the mass spectrometry XML interchange schema
was
successfully demonstrated and the further development of the PSI-MI
standard
discussed.</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal"><b>Conference Session </b></p>
<p class="MsoNormal" style="">The conference session heard a
brief introductory talk by Dr Rolf Apweiler (EBI) in which he described
the
history of the Proteomic Standards Initiative and summarised the
progress made
to date in each area. He also touched upon a new area of interest for
the PSI,
that of enzymology, and described a recent start-up meeting held in
Germany.&nbsp;
Weimin Zhu (EBI) further described the development of the PSI-MS XML
standard
interchange format, which was then put into the context of the wider
efforts to
standardise the field of mass spectrometry by Dr Randall Julian (Eli
Lilly).
Sandra Orchard (EBI) announced the release of Level 1 of the PSI-MI
interchange
format, this announcement has also been submitted for publication and
it is
hoped that the paper will be in press in the very near future. Akihisa
Kenmochi
(Nec Corporation, Tsukuba, Japan) described an XML based
proteomics-oriented
markup language designed to accelerate collaboration among researchers,<span
 style="font-size: 10pt; font-family: &quot;Courier New&quot;;"> </span>Veerasamy
Ravichandran (National Institute of Standards and Technology,
Gaithersburg, United States) spoke on the need for data standards for
two-dimensional gel
electrophoresis and Sandhya Xirasagas (Science Applications
International
Corporation, Germantown, MD, United States) discussed the CEBS object
model for
systems biology (CEBS MAGE Sys BioOM). Finally, Chris Taylor (EBI)
talked about
the PSI data integration model, MIAPE, which will be based upon the
PEDRo model
developed by the University of Manchester [1]. It is planned to develop
different sub-domains which will allow the model to handle specific
data types,
for example 2-D electrophoresis gels or HPLC. The model will support
the
various PSI-interchange formats and also a number of ontologies which
will be
developed, in some cases in collaboration with the MGED<span
 style="text-decoration: underline;"> </span><span
 class="MsoCommentReference"><span style="font-size: 8pt;"><a
 class="msocomanchor" id="_anchor_1"
 onmouseover="msoCommentShow('_anchor_1','_com_1')"
 onmouseout="msoCommentHide('_com_1')" href="#_msocom_1"
 language="JavaScript" name="_msoanchor_1"></a><span
 style="display: none;"> </span></span></span>consortium
[2], and then collated for use by research workers.</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal"><b>Workshop Session </b></p>
<p class="MsoNormal">The meeting started with a show of hands, to
decide whether
to split into separate sessions to discuss PPI/mass spectrometry issues
or to
run the workshop as a single meeting. It transpired that very few
people were
interested in only a single aspect of proteomics data and the decision
was
taken to continue as a single meeting.</p>
<p class="MsoNormal">The mass spectrometry session was chaired by
Weimin Zhu and
opened by Kai Runte and Randall Julian undertaking a live demonstration
on the
PSI-MS XML data interchange format, alongside tools which will allow
the user
to</p>
<ul style="margin-top: 0cm;" type="disc">
  <li class="MsoNormal">Convert from mass spectrometry text formats to
PSI-MS XML format</li>
  <li class="MsoNormal">View and browse stored in PSI-MS XML format</li>
  <li class="MsoNormal">Use PSI-MS XML format as a search engine input</li>
</ul>
<p class="MsoNormal" style="margin-left: 18pt;">&nbsp;</p>
<p class="MsoNormal">The standard is being developed in conjunction
with the
American Society for Tests and Measures (ASTM) which is developing a
broader
based XML-format for analytical information exchange [2]. The PSI-MS
schema is
flexible enough to handle a diversity of experiments with a full range
of
experimental descriptors whilst still remaining compliant with the ASTM
model. </p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">Concerns were raised as to the practicality of
storing large
amounts of data in XML format and the length of time it would take to
access
data stored sequentially. It was emphasised that this is an interchange
format,
not a storage medium, and that users should transfer the data into an
appropriate repository. It is also not anticipated that vendors will
replace
their own proprietary binary file format with PSI-MS XML but rather
that they
would align with the XML export format and supply this as an option for
users.
The response of the vendors, to date, has been excellent and it is
hoped that
this will continue into the future.</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">Future plans include working to develop more
converters
capable of supporting peak list formats other than data (.DTA) files,
to
continue to implement an open source search engine from the Institute
for
Systems Biology (ISB) capable of taking the standard peaklist XML
document as
an input file and to develop a standardized output format from search
engines.
Both hardware and search engine vendors will be further encouraged to
support
the newly developed standards, and, in common with other PSI efforts,
users will
be encouraged to participate in the development of controlled
vocabularies. <span class="MsoCommentReference"><span
 style="font-size: 8pt;"><span class="msoIns"><ins cite="mailto:orchard"
 datetime="2003-11-17T08:31"><a class="msocomanchor" id="_anchor_2"
 onmouseover="msoCommentShow('_anchor_2','_com_2')"
 onmouseout="msoCommentHide('_com_2')" href="#_msocom_2"
 language="JavaScript" name="_msoanchor_2"></a></ins></span></span></span>All
developments
will be made public on the SourceForge website <a
 href="http://psidev.sourceforge.net/ms/">http://psidev.sourceforge.net/ms/</a></p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">The PPI session was co-chaired by David Sherman
(University of Bordeaux) and Jérôme Wojcik (Hybrigenics).&nbsp; A
number of issues arose from
the previous presentation of the Level 1 interchange format but, as
with the
PSI-MS XML format, many of these were issues that will ultimately be
addressed
by the database into which the data is deposited. Questions were raised
as to
the mechanism by which new terms will be added to the already existing
controlled vocabularies which describe interaction type, participant
detection
method etc. The GO model was outlined as a potential methodology which
could be
utilised and it was recognised that a firm decision needs to be made on
this at
the next PSI PPI meeting and the process then rapidly put in place and
advertised to the user community. Finally an appeal was made for help
in user
evaluation of the Level 1 schema and assistance in developing Level 2,
in the
further development of tools to process the data when in this format
and in
expanding the current controlled vocabularies, as need arises.</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">The meeting closed with a final appeal for user
participation and an invitation to all interested parties to sign onto
the PSI
mailing lists to be found on the PSI website
(http://psidev.sourceforge.net/).</p>
<p class="MsoNormal">&nbsp;</p>
<ol style="margin-top: 0cm;" start="1" type="1">
  <li class="MsoNormal">Taylor, C.F., Paton, N.W., Garwood, K.L.,
Kirby, P.D., Stead, D.A., Yin, Z., Deutsch, E.W., Selway, L., Walker,
J., Riba-Garcia, I., Mohammed, S., Deery, M.J., Howard, J.A., Dunkley,
T., Aebersold, R., Kell, D.B., Lilley, K.S., Roepstorff, P., Yates,
J.R., Brass, A., Brown, A.J., Cash, P., Gaskell, S.J., Hubbard, S.J.,
Oliver, S.G. A systematic approach to modeling, capturing, and
disseminating proteomics experimental data. <i>Nat Biotechnol.</i>
2003 21(3):247-54.</li>
  <li class="MsoNormal">Ball, C.A., Sherlock, G., Parkinson. H.,
Rocca-Sera, P., Brooksbank, C., Causton, H.C., Cavalieri, D.,
Gaasterland, T., Hingampm P., Holstege, F., Ringwald, M., Spellman, P.,
Stoeckert, C.J. Jr, Stewart, J.E., Taylor, R., Brazma, A., Quackenbush,
J.; Microarray Gene Expression Data (MGED) Society. Standards for
microarray data.<br>
    <i>Science</i>. 2002 Oct 18;298(5593):539</li>
</ol>
<p class="MsoNormal" style="margin-left: 18pt;">&nbsp;</p>
<p class="MsoNormal" align="center" style="text-align: center;">&nbsp;</p>
<p class="MsoNormal" align="center" style="text-align: center;">&nbsp;</p>
<p class="MsoNormal" align="center" style="text-align: center;">&nbsp;</p>
</div>
<div>
<hr class="msocomoff"
 style="margin-left: 0px; margin-right: auto; height: 1px; width: 100%;">Sandra
Orchard, 2003-11-15<br>
</div>
</body>
</html>
