<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>Protein-protein interaction agenda</title>
</head>
<body>
<h3 style="text-align: center;"><a href="http://psidev.sourceforge.net"><img
 alt="Proteomics Standards Initiative"
 src="file:///X:/pro6/psi/doc/images/psi.gif" width="113" height="75"
 border="0" align="left"></a> <a href="http://www.hupo.org/"><img
 alt="HUPO Human Proteome Organisation"
 src="file:///X:/pro6/psi/doc/images/hupo.gif" width="113" height="75"
 border="0" align="right"></a>HUPO Proteomics Standards Initiative</h3>
<h3 style="text-align: center;">3<sup>rd</sup> workshop <br>
</h3>
<h3 style="text-align: center;">Spring 2004<br>
</h3>
<h3><br>
</h3>
<h3>Introduction:</h3>
Since its conception in April 2002, the HUPO Proteomics Standards
Initiative has contributed to the development of community standards
for proteomics in a collaborative and very dynamic way. As of December
2003, the PSI Molecular Interaction standard has been accepted for
publication in Nature Biotechnology, and the PSI Mass Spectrometry
standard is being finalised and already supported by a comprehensive
set of tools. The initial strategy of focussing on limited, feasible
domains
has been rewarded by significant progress in the focal domains within
only 18 months.<br>
<br>
However, a lot remains to be done. Building on the successful
development and quick adoption of the PSI MI Level 1 standard, we now
need to exploit its potential by increasing inter-database
collaboration to maximise database interoperability, and to promote it
by contacting journals and funding agencies and asking them for support
for the standard. <br>
The PSI Mass Spectrometry work group aims to extend its coverage beyond
the peak list to encompass the full chain from the mass spectrometer to
the identification list. Building on this set of standards, strategies
for its publication and acceptance need to be developed.<br>
And finally, we need to intensively work on the integration of
technology- or domain-specific standards into an encompassing standard
for proteomics data, similar to the work of, and in collaboration with
the MGED consortium for the transcriptomics domain.<br>
To address these issues, we plan the third full meeting of the HUPO
Proteomics Standards Initiative in Spring 2004, time and venue will be
announced soon.<br>
<br>
<h3>Meeting structure:</h3>
<ul>
  <li>The meeting will be composed of </li>
  <ul>
    <li>a plenary session (0.5 days) focussing on short status reports,
and overall proteomics data integration.</li>
    <li>1.5 days of separate work group sessions for</li>
  </ul>
  <ul>
    <ul>
      <li>Mass Spectrometry</li>
      <li>Molecular Interactions</li>
      <li>Integration of Proteomics Workflows<br>
      </li>
    </ul>
    <li>0.5 days results presentation, publication and dissemination
discussion</li>
  </ul>
  <li>Overview<br>
    <table cellpadding="2" cellspacing="4" border="1"
 style="width: 100%; text-align: left; background-color: rgb(255, 255, 255);">
      <tbody>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);"><br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Wednesday<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Thursday<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Friday<br>
          </td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">am<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(255, 255, 255);"
 rowspan="1" colspan="1">Arrival and registration<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);"
 rowspan="2" colspan="1"> Work groups<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Work
groups</td>
        </tr>
        <tr>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">pm<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Plenary
session<br>
          </td>
          <td
 style="vertical-align: top; background-color: rgb(51, 255, 51);">Results,
publications and dissemination, closing remarks</td>
        </tr>
      </tbody>
    </table>
    <br>
  </li>
</ul>
<h3>Preliminary agenda:</h3>
This is an open, draft agenda. Please submit both agenda
items and suggested short presentations to hhe@ebi.ac.uk<br>
<ul>
  <li>Plenary session</li>
  <ul>
    <li>Opening remarks<br>
    </li>
  </ul>
  <ul>
    <li>Molecular Interactions status report</li>
    <li>Mass Spectrometry status report</li>
    <li>Integrating Proteomics - status report</li>
  </ul>
  <li>Parallel work groups:<br>
  </li>
  <ul>
    <li>Mass Spectrometry (Weimin Zhu)<br>
    </li>
    <ol>
      <li>Future development plan</li>
      <li>Controlled vocabulary for M/Z data exchange format</li>
      <li>Converters to convert supported vendor proprietary peaklist
formats to</li>
      <ol>
        <li>standard XML format.</li>
        <li>Standard XML format for search engine results</li>
      </ol>
      <li>Search engines to be supported</li>
      <li>XML schema</li>
      <li>Tools: navigation, visulization</li>
      <li>How to adapt PSI-MS standard data formats</li>
      <li>PSI-MS data formats and MIAPE data model.</li>
    </ol>
    <li>Molecular Interactions (Henning Hermjakob)<br>
    </li>
    <ol>
      <li>Status overview</li>
      <li>Update of the PSI MI XML schema</li>
      <li>Controlled vocabularies</li>
      <ol>
        <li>Procedure for maintenance of controlled vocabularies</li>
        <li>Controlled vocabulary for external databases</li>
      </ol>
      <li>Unified identifiers for interactors</li>
      <li> Data exchange: <br>
      </li>
      <ol>
        <li>"Contract" to make all PSI submitted data available to all
databases and end users?</li>
        <li>Approach publishers with this "contract"?</li>
        <li>Coordination of curation?</li>
        <li>Synchronised effort to create reference sets?</li>
      </ol>
    </ol>
    <li>Separation, Identification, data processing - Integrating
Proteomics Workflows (Chris Taylor)</li>
  </ul>
</ul>
<ol>
  <ol>
    <ol>
      <li>Progress to date -- from PEDRo to MIAPE</li>
      <li>The next steps in MIAPE development:
      </li>
      <ol>
        <li>&nbsp;Sample preparation -- Coordination with MGED
        </li>
        <li>&nbsp;2D gel electrophoresis -- data capture, data storage
        </li>
        <li>&nbsp;Modelling columns and other separation techniques
(e.g. FFE)
        </li>
        <li>&nbsp;Integrating mass spectrometry methods and results
        </li>
        <li>&nbsp;Supporting bioinformatics analyses
        </li>
      </ol>
      <li>Tools to make life easier for everyone - convert/import/export</li>
      <li>Ontology development -- what's there already, what's needed
      </li>
      <li>The grand scheme -- an integrated 'omics model for the life
sciences
      </li>
    </ol>
  </ol>
</ol>
<hr>
<address>Henning Hermjakob, hhe@ebi.ac.uk, 2003-Nov-27<br>
</address>
<!-- Created: Mon Oct  7 13:24:01 BST 2002 --><!-- hhmts start --><!-- hhmts end -->
</body>
</html>
