<html><table>
<tr><td>!version:<td> $Revision: 1.1 $
<tr><td>!date:<td>                 Tue Oct 28 15:21:02 CET 2003
<tr><td>!saved-by:<td> luisa
<tr><td>!autogenerated-by:<td> DAG-Edit version 1.311
!
!Gene Ontology definitions
!
<tr><td>term:<td> 1'-phospho-L-histidine
<tr><td>goid:<td> <a name="MI:0174">MI:0174
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0035
<tr><td colspan=2><hr><tr><tr><td>term:<td> 2'-[3-carboxamido-3-(trimethylammonio)propyl]-L-histidine (diphthamide)
<tr><td>goid:<td> <a name="MI:0185">MI:0185
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0040
<tr><td colspan=2><hr><tr><tr><td>term:<td> 2-pyrrolidone-5-carboxylic acid
<tr><td>goid:<td> <a name="MI:0183">MI:0183
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0031
<tr><td colspan=2><hr><tr><tr><td>term:<td> 3'-methyl-L-histidine
<tr><td>goid:<td> <a name="MI:0164">MI:0164
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0073
<tr><td colspan=2><hr><tr><tr><td>term:<td> 3'-phospho-L-histidine
<tr><td>goid:<td> <a name="MI:0175">MI:0175
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0036
<tr><td colspan=2><hr><tr><tr><td>term:<td> 4-hydroxy-L-proline
<tr><td>goid:<td> <a name="MI:0149">MI:0149
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0030
<tr><td colspan=2><hr><tr><tr><td>term:<td> acetylation
<tr><td>goid:<td> <a name="MI:0121">MI:0121
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> adenilate cyclase complementation
<tr><td>goid:<td> <a name="MI:0014">MI:0014
<tr><td>definition:<td> Adenylate cyclase is encoded by the cyaA gene and contains a catalytic domain which can be proteolytically cleaved into two complementary fragments, T25 and T18, which remain associated in the presence of calmodulin in a fully active ternary complex. In the absence of calmodulin, the mixture of the two fragments does not exhibit detectable activity, suggesting that the two fragments do not associate. When expressed in an adenylate cyclase-deficient E. coli strain (E. coli lacks calmodulin or calmodulin-related proteins), the T25 and T18 fragments fused to putative interacting proteins are brought into close association which result in cAMP synthesis. The level of reconstructed adenylate cyclase can be estimated by monitoring the expression of a cAMP dependent reporter gene.
<tr><td>definition_reference:<td> PMID:9576956
<tr><td colspan=2><hr><tr><tr><td>term:<td> affinity chromatography technologies
<tr><td>goid:<td> <a name="MI:0004">MI:0004
<tr><td>definition:<td> This class of approaches is characterised by the use of affinity resins as tools to purify proteins of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as gluthathione resins for GST fusion proteins, metal resins for histidine-tagged proteins.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> aggregation
<tr><td>goid:<td> <a name="MI:0191">MI:0191
<tr><td>definition:<td> Physical association among molecules.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> alanine scanning
<tr><td>goid:<td> <a name="MI:0005">MI:0005
<tr><td>definition:<td> This approach is used to identify the residues that are involved in an interaction. Several variants of the native protein are prepared by mutating each residue of interest to an alanine. The mutated proteins are expressed, tested for proper folding and probed in the binding assay. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> amidation
<tr><td>goid:<td> <a name="MI:0143">MI:0143
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> anti bait coip: anti bait coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0006">MI:0006
<tr><td>definition:<td> A specific antibody for the protein of interest (bait) is available, this is used to generate a high affinity resin to capture the endogenous bait present in a sample.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> anti tag coip: anti tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0007">MI:0007
<tr><td>definition:<td> A specific antibody for the protein of interest is not available, therefore the bait protein is expressed as a hybrid protein fused to a tag peptide/protein for which efficient and specific antibodies or a specific ligand are available.  
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> array technologies
<tr><td>goid:<td> <a name="MI:0008">MI:0008
<tr><td>definition:<td> In this class of methodologies, the molecules to be tested are presented ordered in an array format (typically at high density) on planar supports. The characteristics and chemical nature of the planar support can vary. This format permits the simultaneous assay, in controlled conditions, of several thousand proteins/peptides/nucleic acids for different functions, for instance their ability to bind any given molecule. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> bacterial display
<tr><td>goid:<td> <a name="MI:0009">MI:0009
<tr><td>definition:<td> The protein of interest is presented on the outer membrane of Gram negative bacteria by expressing it as a fusion partner to peptide signals that direct heterologous proteins to the cell surface. For instance, a single chain Fv (scFv) antibody fragment, consisting of the variable heavy and variable light domains from two separate anti-digoxin monoclonal antibodies, was displayed on the outer membrane of Escherichia coli by fusing it to an Lpp-OmpA. Similar systems have also been developed for gram positive bacteria. Fluorescence-activated cell sorting (FACS), is used to specifically select clones displaying a protein binding to scFv-producing cells.
<tr><td>definition_reference:<td> PMID:8248129
<tr><td>definition_reference:<td> PMID:10436088
<tr><td colspan=2><hr><tr><tr><td>term:<td> beta galactosidase complementation
<tr><td>goid:<td> <a name="MI:0010">MI:0010
<tr><td>definition:<td> Beta-galactosidase activity can be used to monitor the interaction of chimeric proteins. Pairs of inactive beta gal deletion mutants are capable of complementing to restore activity when fused to interacting protein partners. Critical to the success of this system is the choice of two poorly complementing mutant moieties, since strongly complementing mutants spontaneously assemble and produce functional beta-gal activity detectable in absence of any fused protein fragment. 
<tr><td>definition_reference:<td> PMID:9237989
<tr><td>definition_reference:<td> PMID:12042868
<tr><td colspan=2><hr><tr><tr><td>term:<td> beta lactamase complementation
<tr><td>goid:<td> <a name="MI:0011">MI:0011
<tr><td>definition:<td> This strategy is based on a protein fragment complementation assay (PCA) of the enzyme TEM-1 beta-lactamase. The approach includes a simple colorimetric in vitro assays using the cephalosporin nitrocefin and assays in intact cells using the fluorescent substrate CCF2/AM. The combination of in vitro colorimetric and in vivo fluorescence assays of beta-lactamase in mammalian cells permits a variety of sensitive and high-throughput large-scale applications.
<tr><td>definition_reference:<td> PMID:12042868
<tr><td colspan=2><hr><tr><tr><td>term:<td> binding site
<tr><td>goid:<td> <a name="MI:0117">MI:0117
<tr><td>definition:<td> A sequence range within a protein identified as involved in an interaction. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> biophysical
<tr><td>goid:<td> <a name="MI:0013">MI:0013
<tr><td>definition:<td> The application of physical principles and methods to biological experiments
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> bret: bioluminescence resonance energy transfer
<tr><td>goid:<td> <a name="MI:0012">MI:0012
<tr><td>definition:<td> In this variation of the FRET assay the donor fluorophore is replaced by a luciferase (typically Renilla luciferase). In the presence of its substrate, the luciferase catalyses a bioluminescent reaction that excites the acceptor fluorophore through a resonance energy transfer mechanism. As with FRET the energy transfer occurs only if the protein fused to the luciferase and the one fused to the acceptor fluorophore are in close proximity (10-100 Angstrom)
<tr><td>definition_reference:<td> PMID:9874787
<tr><td>definition_reference:<td> PMID for application instance:10725388
<tr><td colspan=2><hr><tr><tr><td>term:<td> cd: circular dichroism
<tr><td>goid:<td> <a name="MI:0016">MI:0016
<tr><td>definition:<td> Circular dichroism (CD) is observed when optically active molecules absorb left and right hand circularly polarized light slightly differently. Linearly polarized light can be viewed as a superposition of two components of circularly polarized light of equal amplitude and phase but opposite handness. When this light passes through an optically active sample the two polarized components are absorbed differently. The difference in left and right handed absorbance A(l)- A(r) is the signal registered in CD spectra. This signal displays distinct features corresponding to different secondary structures present in peptides, proteins and nucleic acids. The analysis of CD spectra can therefore yield valuable information about the secondary structure of biological macromolecules and the interactions among molecules that influence their structure.
<tr><td>definition_reference:<td> PMID:11578931
<tr><td colspan=2><hr><tr><tr><td>term:<td> classical fluorescence spectroscopy
<tr><td>goid:<td> <a name="MI:0017">MI:0017
<tr><td>definition:<td> Proteins contain endogenous fluorophores such as tryptophan residue and heme or flavins groups. Protein folding and protein-protein interaction can be studied by monitoring changes in the tryptophan environment detected by changes in its intrinsic fluorescence. Changes in the fluorescence emission spectrum on complex formation can occur either due to a shift in the wavelength of maximum fluorescence emission or by a shift in fluorescence intensity caused by the mixing of two proteins. The interaction of two proteins causes a shift in the fluorescence emission spectrum relative to the sum of the individual fluorescence spectra, resulting in a difference spectrum [F (complex)-2 F (sum)], which is a measurable effect of the interaction. Loss of fluorescence signal from a substrate can be used to measure protein cleavage.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> coip: coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0019">MI:0019
<tr><td>definition:<td> In this approach an antibody, specific for the protein of interest (bait) or any tag expressed within a fusion protein, is used to separate the bait from a protein mixture or a cell lysate and to capture its ligand simultaneously. The protein partners that bind to the bait protein retained by the resin can then be eluted and identified. 
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> colocalization by electron microscopy
<tr><td>goid:<td> <a name="MI:0020">MI:0020
<tr><td>definition:<td> During the treatment for microscope analysis a tissue section is incubated with high-specificity antibodies coupled to heavy metals (gold). Any tissue section can then be analysed by electron microscopy to localise the target proteins within the cell.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> colocalization by fluorescent probes cloning
<tr><td>goid:<td> <a name="MI:0021">MI:0021
<tr><td>definition:<td> Two proteins can be localised to cell compartments, in the same experiment, if they are expressed as chimeric proteins fused to distinct proteins fluorescing at different wavelengths (Green Fluorescent Protein and Red Fluorescent Protein for example). Using a confocal microscope the two proteins can be visualized in living cells and it can be determined whether they have the same subcellular location. Fluorescence microscopy of cells expressing a GFP fusion protein can also demonstrate dynamic processes such as its translocation from one subcellular compartment to another.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> colocalization by immunostaining
<tr><td>goid:<td> <a name="MI:0022">MI:0022
<tr><td>definition:<td> The subcellular location of a protein can be demonstrated by treating cells fixed on a microscope slide with an antibody specific for the protein of interest. A secondary antibody conjugated with a reactive enzyme (e.g. horseradish peroxidase) is then added. Following a washing step to remove the unbound secondary ligand, a chromogenic substrate (e.g. 3,3', 5,5' tetramethyl benzidine chromogen [TMB]) is converted to a soluble colored product by the conjugated enzyme and can then be visualised by standard microscopic techniques.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> colocalization/visualisation technologies
<tr><td>goid:<td> <a name="MI:0023">MI:0023
<tr><td>definition:<td> Techniques enabling the identification of the subcellular localisation of a protein or complex. Two different proteins show a similar distribution in the cell are said to co-localise.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> confirmational text mining
<tr><td>goid:<td> <a name="MI:0024">MI:0024
<tr><td>definition:<td> Text mining is used to support interactions which have been determined by other methods.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> copurification
<tr><td>goid:<td> <a name="MI:0025">MI:0025
<tr><td>definition:<td> Approaches designed to separate cell components on the basis of their physicochemical properties. The observation that two or more proteins copurify in one or several conditions is taken as an indication that they form a molecular complex.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> correlated mutations
<tr><td>goid:<td> <a name="MI:0026">MI:0026
<tr><td>definition:<td> Pairs of multiple alignments of orthologous sequences are used to identify potential interacting partners as proteins that show covariation of their residue identities between different species. Proteins displaying inter-protein correlated mutations during evolution are likely to be interacting proteins due to co-adapted evolution of their protein interacting interfaces. 
<tr><td>definition_reference:<td> PMID:11933068
<tr><td colspan=2><hr><tr><tr><td>term:<td> cosedimentation
<tr><td>goid:<td> <a name="MI:0027">MI:0027
<tr><td>definition:<td> Separation of a protein mixture under the influence of artificial gravity. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> cosedimentation in solution
<tr><td>goid:<td> <a name="MI:0028">MI:0028
<tr><td>definition:<td> The ultracentrifuge can be used to characterise and/or purify macromolecules in solution according to their mass and hydrodynamic properties. Sedimentation studies provide information about the molecular weight and shape of a molecule. It is also possible to measure the association state of the sample. Both the mass of a molecule and its shape, that influences the friction forces and diffusion that counterbalances gravity, determine the sedimentation speed. 
<tr><td>definition_reference:<td> PMID:10410796
<tr><td colspan=2><hr><tr><tr><td>term:<td> cosedimentation through density gradients
<tr><td>goid:<td> <a name="MI:0029">MI:0029
<tr><td>definition:<td> Sedimentation through a density gradient measures the sedimentation rate of a mixture of proteins through either a glycerol or sucrose gradient. Two interacting proteins will sediment mostly as a complex at concentrations above the binding constant. By varying the concentration of one or both of the complex constituents and taking into account the dilution of the species during sedimentation, one can reasonably accurately estimate the binding constant.
<tr><td>definition_reference:<td> PMID:10410796
<tr><td colspan=2><hr><tr><tr><td>term:<td> cross-linking studies
<tr><td>goid:<td> <a name="MI:0030">MI:0030
<tr><td>definition:<td> Analysis of complexes obtained by chemical treatments that promote the formation of covalent bonds among molecules in close proximity.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> cross-linking with a bifunctional reagent
<tr><td>goid:<td> <a name="MI:0031">MI:0031
<tr><td>definition:<td> A cross-linker is a bifunctional molecule having two reactive ends linked by a spacer, often containing a disulfide bond. Cross-linkers induce the formation of covalent bonds among proteins that are neighbors. When a reducing agent is added the disulfide bridge is cleaved, the cross-linked pairs are released and can be identified. There are various classes of cross-linkers, the most common are those having photoreactive groups that become reactive fluorophores when activated by UV light thereby resulting in photolabeling the cross-linked moieties. 
<tr><td>definition_reference:<td> PMID:10679368
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> de novo protein sequencing by mass spectrometry
<tr><td>goid:<td> <a name="MI:0032">MI:0032
<tr><td>definition:<td> The strategy to determine the complete amino acid sequence of a protein by mass spectrometry relies on the generation of a nested set of fragments differing by one amino acid. This permits to reveal the identity of the residue that has been removed at each degradation step by measuring the mass difference of fragments differing of one residue. Peptide fragments can be obtained by protease treatment combined with the fragmentation promoted by collision (or other methods) within a tandem mass spectrometer. This approach can be carried out with LC MS/MS (Liquid Chromatography Tandem Mass Spectrometry), nanoESI MS/MS (nanoElectrospray Ionisation tandem mass spectrometry), or FTMS (Fourier Transform mass spectrometry) instruments.
<tr><td>definition_reference:<td> PMID:10984529
<tr><td colspan=2><hr><tr><tr><td>term:<td> dehydrofolate reductase reconstruction
<tr><td>goid:<td> <a name="MI:0111">MI:0111
<tr><td>definition:<td> The gene for DHFR is rationally dissected into two fragments called F[1,2] and F[3]. Two proteins or protein domains that are thought to bind to each other can then be fused to either of the two DHFR fragments. Reconstitution of enzyme activity can be monitored in vivo by cell survival in DHFR-negative cells grown in the absence of nucleotides. A fluorescence assay can also be carried out taking advantage of fMTX (the third element of the so called triple hybrid essay) binding to reconstituted DHFR. The basis of this assay is that complementary fragments of DHFR, when expressed and reassembled in cells, will bind with high affinity (Kd 5 540 pM) to fMTX in a 1:1 complex. fMTX is retained in cells by this complex, whereas the unbound fMTX is actively and rapidly transported out of the cells. Survival depends only on the number of molecules of DHFR reassembled. 
<tr><td>definition_reference:<td> PMID:10318894
<tr><td colspan=2><hr><tr><tr><td>term:<td> deletion analysis
<tr><td>goid:<td> <a name="MI:0033">MI:0033
<tr><td>definition:<td> In this approach, once a protein is demonstrated to participate in an interaction, several deletion derivatives are produced and tested in the binding assay to identify the minimal fragment (domain) that can still support the interaction.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> display technologies
<tr><td>goid:<td> <a name="MI:0034">MI:0034
<tr><td>definition:<td> All the methods that permit the physical linking of a protein/peptide to its coding sequence. As a consequence affinity purification of the displayed peptide results in the genetic enrichment of its coding sequence. By these technologies genes encoding a peptide with desired binding properties can be selected over an excess of up to 1012 unrelated molecules.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> docking
<tr><td>goid:<td> <a name="MI:0035">MI:0035
<tr><td>definition:<td> Predicts the structure of a protein-protein complex from the unbound structures of its components.  The initial approach in the majority of docking procedures is based largely on the 'rigid-body' assumption, whereby the proteins are treated as solid objects. Initial scoring of a complex is based on geometric fit or surface complementarity. This generally requires some knowledge of the binding site to limit the number of solutions.
<tr><td>definition_reference:<td> PMID:9631301
<tr><td>definition_reference:<td> PMID for application instance:11478868
<tr><td colspan=2><hr><tr><tr><td>term:<td> domain fusion
<tr><td>goid:<td> <a name="MI:0036">MI:0036
<tr><td>definition:<td> The rosetta stone, or domain fusion procedure, is based on the assumption that proteins whose homologues in other organisms happen to be fused into a single protein chain are likely to interact or to be functionally related. 
<tr><td>definition_reference:<td> PMID:10573422
<tr><td colspan=2><hr><tr><tr><td>term:<td> domain profile pairs
<tr><td>goid:<td> <a name="MI:0037">MI:0037
<tr><td>definition:<td> This approach use a protein interaction network of a given organism to infer interaction in another organism using information about the interacting region. The regions or domains involved in interactions are clustered if they share sequence similarity and have common interacting partners. The resulting domain profiles are then used to screen the proteome of another organism and domain-domain interactions are inferred. Ultimately, an inferred protein interaction map is built in this second organism. 
<tr><td>definition_reference:<td> PMID:11473021
<tr><td colspan=2><hr><tr><tr><td>term:<td> dynamic light scattering
<tr><td>goid:<td> <a name="MI:0038">MI:0038
<tr><td>definition:<td> In dynamic light scattering, particle diffusion in solution gives rise to fluctuations in the intensity of the scattered light on the microsecond scale. The hydrodynamic radius of the particles can be easily calculated.
<tr><td>definition_reference:<td> PMID:9013660
<tr><td colspan=2><hr><tr><tr><td>term:<td> edman degradation
<tr><td>goid:<td> <a name="MI:0039">MI:0039
<tr><td>definition:<td> In this procedure the N-terminus amino acid is cleaved from a polypeptide and identified by high-pressure liquid chromatography. The cycle is repeated on the evershortening polypeptide until all the residues are identified. On average only 20-30 consecutive cycles can be performed and lead to amino acid identification. Longer polypeptides or full length proteins must be cleaved by specific protease before Edman degradation and their sequences built by fragment overlapping. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> electron microscopy
<tr><td>goid:<td> <a name="MI:0040">MI:0040
<tr><td>definition:<td> Electron microscopy methods provide insights into the structure of biological macromolecules and their supramolecular assemblies. Resolution is on average around 10 Angstron but can reach the atomic level when the samples analysed are 2D crystals. Different types of samples can be analysed by electron microscopy: crystals, single particles like viruses, macromolecular complexes or entire cells and tissue sections. Samples can be chemically fixed or vitrified by rapid freezing in liquid ethane, and then transferred into the electron microscope. Data collection consists of the recording of electron diffraction data (2D crystals) and images. Depending on the type of sample, different approaches are used to analyse and merge images and electron diffraction data. 
<tr><td>definition_reference:<td> PMID:11785754
<tr><td colspan=2><hr><tr><tr><td>term:<td> electron resonance
<tr><td>goid:<td> <a name="MI:0043">MI:0043
<tr><td>definition:<td> A form of spectroscopy in which the absorption of microwave by a sample in a strong magnetic field is used to study atoms or molecules with unpaired electrons.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> elisa: enzyme-linked immunosorbent assay
<tr><td>goid:<td> <a name="MI:0044">MI:0044
<tr><td>definition:<td> Following non-covalent binding of a purified primary ligand to a solid phase, a blocking reagent is added to prevent any non-specific binding. A specific antigen is then allowed to bind to the primary ligand. Unbound antigen is removed by washing and a secondary antibody conjugated to an enzyme (e.g. horseradish peroxidase) is added. Following a washing step to remove unbound secondary ligand, the extent to which a chromogenic substrate (e.g. 3,3', 5,5' tetramethyl benzidine chromogen [TMB]) is converted to a soluble colored product by the conjugated enzyme in a given time is determined by spectrophotometry using a standard microplate absorbance reader. A similar type of approach can be utilized to detect enzymatic activities. The substrate, attached to a solid phase is incubated in the presence of the enzyme and the enzymatic modification is monitored by an antibody that is specific for the modified substrate (for instance a phosphorylated protein).
<tr><td>definition_reference:<td> PMID:11906746
<tr><td colspan=2><hr><tr><tr><td>term:<td> endor: electron nuclear double resonance
<tr><td>goid:<td> <a name="MI:0041">MI:0041
<tr><td>definition:<td> A combination of NMR and EPR. The lines in the EPR spectrum that are caused by coupling of an unpaired electron nearby nuclei change in intensity when these nuclei are excited at their NMR frequency.
<tr><td>definition_reference:<td> PMID:11817959
<tr><td>definition_reference:<td> PMID:11988476
<tr><td>definition_reference:<td> PMID for application instance:12186859
<tr><td colspan=2><hr><tr><tr><td>term:<td> epr: electron paramagnetic resonance
<tr><td>goid:<td> <a name="MI:0042">MI:0042
<tr><td>definition:<td> EPR (also called ESR, Electron Spin Resonance) spectroscopy is analogous to NMR, but is based on the excitation of unpaired electrons instead of nuclei. Unpaired (single) electron are only found in radicals and some metal ions (paramagnetic species); the EPR spectrum provides information about the environment and mobility of the paramagnetic species. The magnetic interaction of two paramagnetic centers in a protein can be used to calculate the distance between them; this allows studies of the movements and interactions of protein segments. In proteins without any intrinsic unpaired electrons it is possible to attach a radical probe (spin label). Stable nitroxide radicals can be bound to amino acid residues, in analogy with fluorescent probes. In combination with site directed mutagenesis this method is used in particular to study structure and assembly of membrane proteins, by measuring with EPR whether an amino acid is in a polar or non polar environment. 
<tr><td>definition_reference:<td> PMID:11817959
<tr><td colspan=2><hr><tr><tr><td>term:<td> experimental
<tr><td>goid:<td> <a name="MI:0045">MI:0045
<tr><td>definition:<td> Methods based on laboratory experiments.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> experimental knowledge based
<tr><td>goid:<td> <a name="MI:0046">MI:0046
<tr><td>definition:<td> Predictive algorithms that rely on the information obtained by experimental results
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> facs: fluorescence-activated cell sorting
<tr><td>goid:<td> <a name="MI:0054">MI:0054
<tr><td>definition:<td> Cells in suspension flow through a laser beam, the scattered light or emitted fluorescence is measured, filtered and converted to digital values. Cells can be sorted according to their properties. Using flow cytometry, any fluorescent or light scattering experiment can be carried out on entire cells. With this instrument, interactions occurring either on cell surfaces or in any other subcellular location can be studied by using suitable fluorescent labels. 
<tr><td>definition_reference:<td> PMID:11988464
<tr><td colspan=2><hr><tr><tr><td>term:<td> far western blotting
<tr><td>goid:<td> <a name="MI:0047">MI:0047
<tr><td>definition:<td> Proteins are fractionated by PAGE (SDS-polyacrylamide gel electrophoresis), transferred to a nitrocellulose membrane and tested for the ability to bind to a protein, a peptide, or any other ligand. Cell lysates can also be fractionated before gel electrophoresis to increase the sensitivity of the method for detecting interactions with rare proteins. Denaturants are removed during the blotting procedure, which allows many proteins to recover (or partially recover) activity. However, if biological activity is not recoverable, the proteins can be fractionated by a non denaturing gel system. This variation of the method eliminates the problem of activity regeneration and allows the detection of binding when the presence of a protein complex is required for binding. The protein probe can be prepared by any one of several procedures, while fusion affinity tags greatly facilitate purification. Synthesis in E. coli with a GST fusion, epitope tag, or other affinity tag is most commonly used. The protein of interest can then be radioactively labelled, biotinylated, or used in the blotting procedure as an unlabeled probe that is detected by a specific antibody.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> fcs: fluorescence correlation spectroscopy
<tr><td>goid:<td> <a name="MI:0052">MI:0052
<tr><td>definition:<td> FCS monitors the random motion of fluorescently labelled molecules inside a defined volume irradiated by a focused laser beam. These fluctuations provide information on the rate of diffusion or diffusion time of a particle and this is directly dependent on the particle mass. As a consequence, any increase in the mass of a biomolecule, e.g. as a result of an interaction with a second molecule, is readily detected as an increase in the diffusion time of the particle. From these results the concentration of the different molecules can be calculated as well as their binding constant.
<tr><td>definition_reference:<td> PMID:10733953
<tr><td colspan=2><hr><tr><tr><td>term:<td> feature detection
<tr><td>goid:<td> <a name="MI:0003">MI:0003
<tr><td>definition:<td> Method to determine the features of the proteins involved in the interaction.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> feature type
<tr><td>goid:<td> <a name="MI:0116">MI:0116
<tr><td>definition:<td> Property of a subsequence that interferes with the binding of the protein
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> filamentous phage display
<tr><td>goid:<td> <a name="MI:0048">MI:0048
<tr><td>definition:<td> Filamentous phages (M13, f1, fd) have been extensively used to develop and implement the technology of phage display. Repertoires of relatively short peptides of random amino acid sequences or cDNA libraries have been constructed and searched successfully. Most experiments have taken advantage of the ability to assemble phages decorated with hybrid versions of the receptor protein pIII or of the major coat protein pVIII. Both systems allow the display of foreign peptides by fusion to the amino-terminus of the capsid protein but differ in the number of peptide copies that can be displayed on each phage particle. Display libraries of very diverse protein fragments have been constructed by fusing either genomic or cDNA fragments to gene III or gene VIII.
<tr><td>definition_reference:<td> PMID:7682645
<tr><td colspan=2><hr><tr><tr><td>term:<td> filter binding
<tr><td>goid:<td> <a name="MI:0049">MI:0049
<tr><td>definition:<td> Protein expressed by different clones of an expression library are bound to a nitrocellulose membrane, by colony (bacterial library) or plaque (phage library) blotting. A labelled protein can then be used as a probe to identify clones expressing proteins that interact with the probe. Interactions occur on nitrocellulose filters. The method is highly general and therefore widely applicable, in that proteins as diverse as transcription factors and growth factor receptors have been successfully used as probes. A variety of approaches can be used to label the protein ligand, alternatively the ligand can be detected by a specific antibody.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> flag tag coip: flag tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0050">MI:0050
<tr><td>definition:<td> The protein of interest is expressed as a fusion to the peptide DYKDDDDKV for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> fluorescence technologies
<tr><td>goid:<td> <a name="MI:0051">MI:0051
<tr><td>definition:<td> Techniques based upon the measurement of the emission of one or more photons by a molecule activated by the absorption of a quantum of electro-magnetic radiation. Typically the emission, which is characterised by a wavelength that is longer than the one of excitatory radiation, occurs within 10-8 seconds.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> formylation
<tr><td>goid:<td> <a name="MI:0146">MI:0146
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> fps: fluorescence polarization spectroscopy
<tr><td>goid:<td> <a name="MI:0053">MI:0053
<tr><td>definition:<td> Because of the long lifetimes of excited fluorescent molecules (nanoseconds), fluorescence can be used to monitor the rotational motion of molecules, which occurs on this timescale. This is accomplished experimentally by excitation with plane-polarized light, followed by measurement of the emission at parallel and perpendicular planes. Since rotational correlation times depend on the size of the molecule, this method can be used to measure the binding of two proteins because the observed polarization increase when a larger complex is formed. A fluorescence anisotropy experiment is normally carried out with a protein bearing a covalently added fluorescent group, which increases both the observed fluorescence lifetime of the excited state and the intensity of the fluorescent signal. Residue modification can be assessed by addition of an antibody which binds to the modified residue and alters the molecular weight of the complex. A variation of this technique has been used to show interaction of a DNA binding protein with another protein.  In this case the DNA rather than protein is fluorescently labelled.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td>definition_reference:<td> PMID:12805227
<tr><td colspan=2><hr><tr><tr><td>term:<td> fret: fluorescent resonance energy transfer
<tr><td>goid:<td> <a name="MI:0055">MI:0055
<tr><td>definition:<td> FRET is a quantum mechanical process involving the radiationless transfer of energy from a donor fluorophore to an appropriately positioned acceptor fluorophore. The fluorophores are genetically fused to the protein in analysis and cotransfected. Three basic conditions must be fulfilled for FRET to occur between a donor molecule and acceptor molecule. First, the donor emission spectrum must significantly overlap the absorption spectrum of the acceptor. Second, the distance between the donor and acceptor fluorophores must fall within the range 0.002 to 0.01 10-6m. Third, the donor and acceptor fluorophores must be in favourable orientations.
<tr><td>definition_reference:<td> PMID:11558993
<tr><td colspan=2><hr><tr><tr><td>term:<td> full identification by sequencing
<tr><td>goid:<td> <a name="MI:0056">MI:0056
<tr><td>definition:<td> The DNA to be sequenced is used as template for the in vitro synthesis, by DNA polymerase, of a set of partial replicas, all beginning at the same place, but terminating at different points along the DNA chain. The key to this method is the use of dideoxyribonucleoside triphosphates in which the deoxyribose 3'-OH group present in normal nucleotides is missing; when such a modified nucleotide is incorporated into a DNA chain, it blocks the elongation of the chain. To determine the full sequence, the four different chain-terminating dideoxyribonucleosides are used in competition with an excess of deoxyribonucleosides in separate DNA synthesis reactions on the same DNA template. When the products of these four reactions are analysed by electrophoresis in four parallel lanes of a denaturing polyacrylamide gel, the DNA sequence can be derived. Every lane displays a family of DNA fragments of different lengths, reflecting the different sites at which a specific residue occurs in the original DNA.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> gene neighbourhoods
<tr><td>goid:<td> <a name="MI:0057">MI:0057
<tr><td>definition:<td> Gene pairs that show a conserved topological neighbourhood in many prokaryotic genomes are considered by this approach to encode interacting or functionally related proteins. By measuring the physical distance of any given gene pair in different genomes, interacting partners are inferred
<tr><td>definition_reference:<td> PMID:9787636
<tr><td colspan=2><hr><tr><tr><td>term:<td> genome based prediction
<tr><td>goid:<td> <a name="MI:0058">MI:0058
<tr><td>definition:<td> Methods that require fully sequenced genomes either because they are based on the comparison of genome topology or on the identification of orthologous sequences in different genomes.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> gst pull down
<tr><td>goid:<td> <a name="MI:0059">MI:0059
<tr><td>definition:<td> The bait protein is expressed and purified as a fusion to the gluthathione S-tranferase protein. The bait protein is normally attached to a gluthathione sepharose resin or alternatively to a support containing an anti-GST antibody.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> ha tag coip: ha tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0060">MI:0060
<tr><td>definition:<td> The protein of interest is expressed as a fusion to the peptide YPYDVPDYA (a fragment of the influenza hemagglutinin protein) for which antibodies are commercially available 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> his pull down
<tr><td>goid:<td> <a name="MI:0061">MI:0061
<tr><td>definition:<td> The bait protein is expressed and purified fused to an amino or carboxyterminal tail containing a variable number of histidines. The bait protein is normally attached to a metal (usually nickel) resin.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> his tag coip: his tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0062">MI:0062
<tr><td>definition:<td> The protein of interest is expressed as a fusion to a poly-His tail. This permits purification by chromatography over a metal column or by binding to commercially available anti poly-His antibodies.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> hotspot
<tr><td>goid:<td> <a name="MI:0119">MI:0119
<tr><td>definition:<td> Residue of a protein whose identity modifies significantly interaction properties
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> hydroxylation
<tr><td>goid:<td> <a name="MI:0148">MI:0148
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> in silico
<tr><td>goid:<td> <a name="MI:0063">MI:0063
<tr><td>definition:<td> Computational methods to predict an interaction.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> interaction detection
<tr><td>goid:<td> <a name="MI:0001">MI:0001
<tr><td>definition:<td> Method to determine the interaction.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> interaction type
<tr><td>goid:<td> <a name="MI:0190">MI:0190
<tr><td>definition:<td> Connection between proteins
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> interologs mapping
<tr><td>goid:<td> <a name="MI:0064">MI:0064
<tr><td>definition:<td> Protein interactions, experimentally detected in an organism, are extended to a second organism assuming that homologue proteins, in different organisms, maintain their interaction properties.
<tr><td>definition_reference:<td> PMID:11731503
<tr><td colspan=2><hr><tr><tr><td>term:<td> itc: isothermal titration calorimetry
<tr><td>goid:<td> <a name="MI:0065">MI:0065
<tr><td>definition:<td> Isothermal titration calorimetry (ITC) measures directly the energy associated with a chemical reaction triggered by the mixing of two components. A typical ITC experiment is carried out by the stepwise addition of one of the reactants (~10-6 L per injection) into the reaction cell (~1mL) containing the second reactant. The chemical reaction occurring after each injection either releases or absorbs heat (qi) proportional to the amount of ligand that binds to the protein with a characteristic binding enthalpy (DH). As modern ITC instruments operate on the heat compensation principle, the instrumental response (measured signal) is the amount of power (microcalories per second) necessary to maintain constant the temperature difference between the reaction and the reference cells. Because the amount of uncomplexed protein available progressively decreases after each successive injection, the magnitude of the peaks becomes progressively smaller until complete saturation is achieved. The difference between the concentration of bound ligand in the ith and (i-1)th injections depends on the binding constant Ka and the total ligand injected. The calculations depend on the binding model (number of substrates). Analysis of the data yields DH and DG = -RTlnKa. The entropy change is obtained by using the standard thermodynamic expression DG = DH-TDS.
<tr><td>definition_reference:<td> PMID:11785756
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-3-oxoalanine
<tr><td>goid:<td> <a name="MI:0182">MI:0182
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0185
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-alanine amide
<tr><td>goid:<td> <a name="MI:0144">MI:0144
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0081
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-arginine amide
<tr><td>goid:<td> <a name="MI:0145">MI:0145
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0082
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-aspartic 4-phosphoric anhydride
<tr><td>goid:<td> <a name="MI:0172">MI:0172
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0033
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-beta-methylthioaspartic acid
<tr><td>goid:<td> <a name="MI:0161">MI:0161
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0232
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-glutamic acid 5-methyl ester
<tr><td>goid:<td> <a name="MI:0163">MI:0163
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0072
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-glutamyl 5-glycerylphosphorylethanolamine
<tr><td>goid:<td> <a name="MI:0184">MI:0184
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0170
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-selenocysteine
<tr><td>goid:<td> <a name="MI:0180">MI:0180
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0022
<tr><td colspan=2><hr><tr><tr><td>term:<td> L-selenomethionine
<tr><td>goid:<td> <a name="MI:0181">MI:0181
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> lambda phage display
<tr><td>goid:<td> <a name="MI:0066">MI:0066
<tr><td>definition:<td> Morphologically classified as one of the siphoviridae, lambda is a temperate bacteriophage of E.coli, with a double-stranded DNA genome. It has an icosahedral head attached to a flexible helical tail. Both the tail protein pV and the head protein pD have been used for displaying (C or N terminally) foreign peptides on the viral capsid.
<tr><td>definition_reference:<td> PMID:7682645
<tr><td colspan=2><hr><tr><tr><td>term:<td> light scattering
<tr><td>goid:<td> <a name="MI:0067">MI:0067
<tr><td>definition:<td> Dynamic and static laser light scattering probes the size, shape, and structure of biological macromolecules or of their assemblies. A beam is focused on an optically clear cylindrical cell containing the sample. Most of the light passes directly through the sample. A small portion of the light is scattered; the scattered light intensity containing information about the scattering particle is detected at an angle (typically in the range 15-180degrees) from the direction of the incident beam. 
<tr><td>definition_reference:<td> PMID:9013660
<tr><td colspan=2><hr><tr><tr><td>term:<td> lipid modification
<tr><td>goid:<td> <a name="MI:0150">MI:0150
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> mass detection of residue modification
<tr><td>goid:<td> <a name="MI:0068">MI:0068
<tr><td>definition:<td> Mass spectrometry can be used to characterise chemical modifications within peptides. One approach consists in the observation of a mass difference when a sample is treated with an enzyme that can specifically remove a peptide modification, for instance a phosphatase. The mass difference corresponds to the mass of the chemical group covalently linked to a residue. Such experiments carried out with a MALDI-TOF (Matrix-assisted laser desorption ionization time-of-flight ) do not allow the mapping of the modification site within the sequence, whereas any tandem mass spectrometer (LC MS/MS Liquid Chromatography Tandem Mass Spectrometry, nanoESI MS/MS nanoElectrospray Ionisation tandem mass spectrometry, FTMS Fourier Transform mass spectrometry) provide such information. A second approach consists of the direct mass measurement of the ionized chemical group dissociated from the residue within a tandem mass spectrometer. Both approaches need a prior enrichment of the modified peptide population in the samples with IMAC (Immobilized Metal Affinity Chromatography)or specific anti-modification antibodies.
<tr><td>definition_reference:<td> PMID:11395414
<tr><td>definition_reference:<td> PMID for application instance:11875433
<tr><td colspan=2><hr><tr><tr><td>term:<td> mass spectrometry studies of complexes
<tr><td>goid:<td> <a name="MI:0069">MI:0069
<tr><td>definition:<td> Mass spectrometric approaches to the study of protein in complexes permits the identification of subunit stoichiometry and transient associations. By preserving complexes intact in the mass spectrometer, mass measurement can be used for monitoring changes in different experimental conditions, or to investigate how variations of collision energy affect their dissociation. Complexes can be transferred into the gas phase by a nanoflow ESI (Electrospray Ionisation) ionisation device. This is the method of choice for the investigation of the higher-order structure of biomolecules because it allows direct analysis of dilute aqueous solutions and the desolvation process is efficient and closer to the native-like solution environment. Mass measurements of intact macromolecular complexes is largely the domain of time of-flight (TOF) MS. This is mostly due to the high sensitivity and speed of TOF analysis, as well as the virtually unlimited m/z range. Quadrupole TOF (Q-TOF) type mass spectrometers combine a quadrupole mass filter with an orthogonal TOF analyser. These spectrometers provide supplementary structural information for ions isolated in the quadrupole and analysed in the TOF. This allows complexes well in excess of 60 kDa to be dissociated and consequently their subunit composition can be determined.
<tr><td>definition_reference:<td> PMID:12504676
<tr><td>definition_reference:<td> PMID for application instance:12057199
<tr><td colspan=2><hr><tr><tr><td>term:<td> methylation
<tr><td>goid:<td> <a name="MI:0157">MI:0157
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> mobility shift
<tr><td>goid:<td> <a name="MI:0070">MI:0070
<tr><td>definition:<td> Protein modifications can be identified by gel electrophoresis since any change in the mass and/or the charge of the protein can alter its mobility in PAGE. Although this method does not allow the unequivocally identification of the type of modification that has caused the shift, it is possible, by combining this approach with more direct methods, to correlate the extent of the shift to a specific modification.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> molecular sieving
<tr><td>goid:<td> <a name="MI:0071">MI:0071
<tr><td>definition:<td> In sizing columns (gel filtration), the elution position of a protein or of a complex depends on its Stokes radius. Molecules with a radius that is smaller than the bead size are retained and retarded by the interaction with the matrix. The observation that two proteins, loaded on a sieving column, elute in a fraction(s) corresponding to a MW that is larger than the MW of either protein may be taken as an indication that the two proteins interact. Furthermore this technique provides a conceptually simple method for evaluating the affinity of the interaction.
<tr><td>definition_reference:<td> PMID:7708014
<tr><td colspan=2><hr><tr><tr><td>term:<td> monoclonal antibody
<tr><td>goid:<td> <a name="MI:0072">MI:0072
<tr><td>definition:<td> monoclonal antibodies are monospecific antibodies produced in the supernatant of a cell line obtained by fusing a lymphocyte B to a myeloma cell line or selected by phage display technology.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> mrna display
<tr><td>goid:<td> <a name="MI:0073">MI:0073
<tr><td>definition:<td> This method relies on the covalent coupling of mRNA to the nascent polypeptide. The mRNA (natural or artificial) is first covalently linked to a short DNA linker carrying a puromycin moiety. The mRNA mixture is then translated in vitro. When the ribosome reaches the RNA-DNA junction the ribosome stalls and the puromycin moiety enters the peptidyltransferase site of the ribosome and forms a covalent linkage to the nascent polypeptide. As a result the protein and the mRNA are covalently joined and can be isolated from the ribosome and purified. In the current protocol, a cDNA strand is then synthesised to form a less sticky RNA-DNA hybrid and these complexes are finally used for affinity selection. As in most display approaches, several selections cycles (3-6) are sufficient to enrich for mRNAs encoding ligand proteins.
<tr><td>definition_reference:<td> PMID:11551470
<tr><td colspan=2><hr><tr><tr><td>term:<td> mutation
<tr><td>goid:<td> <a name="MI:0118">MI:0118
<tr><td>definition:<td> Sequence variation due to insertion, deletion or substitution event
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> mutation analysis
<tr><td>goid:<td> <a name="MI:0074">MI:0074
<tr><td>definition:<td> Several mutant proteins are produced by random or directed techniques and assayed for their ability to support binding. Mutants defective in binding are tested for correct folding to pinpoint the residues that are directly involved in binding.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> myc tag coip : myc tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0075">MI:0075
<tr><td>definition:<td> The protein of interest is expressed as a fusion to the peptide EUKLISEED (a fragment of the Myc oncogene protein) for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> N,N,N-trimethyl-L-alanine
<tr><td>goid:<td> <a name="MI:0159">MI:0159
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0062
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-alanine
<tr><td>goid:<td> <a name="MI:0122">MI:0122
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0041
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-arginine
<tr><td>goid:<td> <a name="MI:0123">MI:0123
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-asparagine
<tr><td>goid:<td> <a name="MI:0124">MI:0124
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-aspartic acid
<tr><td>goid:<td> <a name="MI:0125">MI:0125
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0042
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-cysteine
<tr><td>goid:<td> <a name="MI:0126">MI:0126
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0043
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-glutamic acid
<tr><td>goid:<td> <a name="MI:0128">MI:0128
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0044
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-glutamine
<tr><td>goid:<td> <a name="MI:0127">MI:0127
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0045
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-histidine
<tr><td>goid:<td> <a name="MI:0130">MI:0130
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-isoleucine
<tr><td>goid:<td> <a name="MI:0131">MI:0131
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0047
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-leucine
<tr><td>goid:<td> <a name="MI:0132">MI:0132
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-methionine
<tr><td>goid:<td> <a name="MI:0135">MI:0135
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0049
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-phenylalanine
<tr><td>goid:<td> <a name="MI:0136">MI:0136
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-proline
<tr><td>goid:<td> <a name="MI:0137">MI:0137
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0050
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-serine
<tr><td>goid:<td> <a name="MI:0138">MI:0138
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0051
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-threonine
<tr><td>goid:<td> <a name="MI:0139">MI:0139
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0052
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-tryptophan
<tr><td>goid:<td> <a name="MI:0140">MI:0140
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-tyrosine
<tr><td>goid:<td> <a name="MI:0141">MI:0141
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0053
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetyl-L-valine
<tr><td>goid:<td> <a name="MI:0142">MI:0142
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0054
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-acetylglycine
<tr><td>goid:<td> <a name="MI:0129">MI:0129
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0046
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-formyl-L-methionine
<tr><td>goid:<td> <a name="MI:0147">MI:0147
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0021
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-methyl-L-alanine
<tr><td>goid:<td> <a name="MI:0158">MI:0158
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0061
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-methyl-L-methionine
<tr><td>goid:<td> <a name="MI:0168">MI:0168
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0064
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-methyl-L-phenylalanine
<tr><td>goid:<td> <a name="MI:0169">MI:0169
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0065
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-myristoyl-glycine
<tr><td>goid:<td> <a name="MI:0155">MI:0155
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0059
<tr><td colspan=2><hr><tr><tr><td>term:<td> N-palmitoyl-L-cysteine
<tr><td>goid:<td> <a name="MI:0153">MI:0153
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0060
<tr><td colspan=2><hr><tr><tr><td>term:<td> N2-acetyl-L-lysine
<tr><td>goid:<td> <a name="MI:0133">MI:0133
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0048
<tr><td colspan=2><hr><tr><tr><td>term:<td> N5-methyl-L-glutamine
<tr><td>goid:<td> <a name="MI:0162">MI:0162
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0071
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6,N6,N6-trimethyl-L-lysine
<tr><td>goid:<td> <a name="MI:0167">MI:0167
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0074
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6,N6-dimethyl-L-lysine
<tr><td>goid:<td> <a name="MI:0166">MI:0166
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0075
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-(4-amino-2-hydroxybutyl)-L-lysine (hypusine)
<tr><td>goid:<td> <a name="MI:0187">MI:0187
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0116
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-acetyl-L-lysine
<tr><td>goid:<td> <a name="MI:0134">MI:0134
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0055
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-biotinyl-L-lysine
<tr><td>goid:<td> <a name="MI:0186">MI:0186
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0117
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-methyl-L-lysine
<tr><td>goid:<td> <a name="MI:0165">MI:0165
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0076
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-myristoyl-L-lysine
<tr><td>goid:<td> <a name="MI:0156">MI:0156
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0078
<tr><td colspan=2><hr><tr><tr><td>term:<td> N6-retinal-L-lysine
<tr><td>goid:<td> <a name="MI:0188">MI:0188
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0120
<tr><td colspan=2><hr><tr><tr><td>term:<td> neural network on interface properties
<tr><td>goid:<td> <a name="MI:0076">MI:0076
<tr><td>definition:<td> Neural networks are trained on the properties of residues belonging to a cluster of residues that are neighbors in space on protein surface. The predictor permits the inference of the residues that are likely to be on an interaction interface.
<tr><td>definition_reference:<td> PMID:11455607
<tr><td>definition_reference:<td> PMID:11874449
<tr><td colspan=2><hr><tr><tr><td>term:<td> nmr: nuclear magnetic resonance
<tr><td>goid:<td> <a name="MI:0077">MI:0077
<tr><td>definition:<td> NMR requires a small volume of concentrated protein solution that is placed in a strong magnetic field. Certain atomic nuclei, and in particular hydrogen, have a magnetic moment or spin; that is, they have an intrinsic magnetisation, like a bar magnet. The spin aligns along the strong magnetic field, but can be changed to a misaligned excited state in response to applied radio frequency (RF) pulses of electromagnetic radiation. When the excited hydrogen nuclei relax to their aligned state, they emit RF radiation, which can be measured and displayed as a spectrum. The nature of the emitted radiation depends on the environment of each hydrogen nucleus, and if one nucleus is excited, it will influence the absorption and emission of radiation by other nuclei that lie close to it. It is consequently possible, by an ingenious elaboration of the basic NMR technique known as two-dimensional NMR, to distinguish the signals from hydrogen nuclei in different amino acid residues and to identify and measure the small shifts in these signals that occur when these hydrogen nuclei lie close enough to interact: the size of such a shift reveals the distance between the interacting pair of hydrogen atoms. In this way NMR can give information about the distances between the parts of the protein molecule. NMR provides information about interacting atoms thereby permitting to obtain information about protein structure and protein-protein interaction. 
<tr><td>definition_reference:<td> PMID:12120505
<tr><td>definition_reference:<td> PMID for application instance:12062432
<tr><td colspan=2><hr><tr><tr><td>term:<td> nucleotide sequence identification
<tr><td>goid:<td> <a name="MI:0078">MI:0078
<tr><td>definition:<td> Identification of a nucleotide sequence. Depending on the experimental design, nucleotide sequence can be determined before the interaction detection while building a collection of clones or after the selection of\nrandomly generated clones. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> O-phospho-L-serine
<tr><td>goid:<td> <a name="MI:0176">MI:0176
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0037
<tr><td colspan=2><hr><tr><tr><td>term:<td> O-phospho-L-threonine
<tr><td>goid:<td> <a name="MI:0177">MI:0177
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0038
<tr><td colspan=2><hr><tr><tr><td>term:<td> O4'-phospho-L-tyrosine
<tr><td>goid:<td> <a name="MI:0178">MI:0178
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0039
<tr><td colspan=2><hr><tr><tr><td>term:<td> omega-N,omega-N-dimethyl-L-arginine
<tr><td>goid:<td> <a name="MI:0160">MI:0160
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0068
<tr><td colspan=2><hr><tr><tr><td>term:<td> omega-N-phospho-L-arginine
<tr><td>goid:<td> <a name="MI:0171">MI:0171
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0222
<tr><td colspan=2><hr><tr><tr><td>term:<td> other biochemical technologies
<tr><td>goid:<td> <a name="MI:0079">MI:0079
<tr><td>definition:<td> Experimental methods that could not be assigned to the other large group of technologies.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> other modification
<tr><td>goid:<td> <a name="MI:0179">MI:0179
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> partial dna sequence identification by hybridization
<tr><td>goid:<td> <a name="MI:0080">MI:0080
<tr><td>definition:<td> Genes are recognised by hybridization of a probe with a fragment of the gene sequence. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> participant detection
<tr><td>goid:<td> <a name="MI:0002">MI:0002
<tr><td>definition:<td> Method to determine the proteins involved in the interaction.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> pca: protein complementation assay
<tr><td>goid:<td> <a name="MI:0090">MI:0090
<tr><td>definition:<td> The function of numerous proteins (enzymes, transcription factors, etc...) can be rationally dissected into two fragments that fold autonomously but cannot complement to reconstitute the protein function, unless they are located in close proximity. In a two hybrid experiment, restoration of the activity by complementation of the two fragments when expressed as fusion with two polypeptides is taken as an evidence that the two polypeptides interact together.
<tr><td>definition_reference:<td> PMID:11495741
<tr><td colspan=2><hr><tr><tr><td>term:<td> peptide array
<tr><td>goid:<td> <a name="MI:0081">MI:0081
<tr><td>definition:<td> The peptide synthesis methods offers numerous opportunities to synthesise and subsequently screen large arrays of synthetic peptides on planar cellulose supports. Discrete spots are arranged as arrays on membrane sheets where each spot is individually accessed by manual or automated delivery of the appropriate reagent solutions. Over the past few years protein-protein recognition, peptide-metal ion interactions, peptide-nucleic acid binding, enzymatic modification of peptides experiments, have been explored using synthetic peptide arrays on planar support.
<tr><td>definition_reference:<td> PMID:11167074
<tr><td colspan=2><hr><tr><tr><td>term:<td> peptide massfingerprinting
<tr><td>goid:<td> <a name="MI:0082">MI:0082
<tr><td>definition:<td> This approach leads to protein identification by matching peptide masses, as measured by mass spectrometry, to the ones calculated from in silico fragmentation of a protein sequence database. A peptide mixture from a tryptic digest is analysed by MALDI-MS (Matrix-assisted laser desorption ionization mass spectrometry). The list of peptide masses obtained by MALDI MS is automatically compared to the calculated masses of the predicted peptide fragments for each entry in the database. High mass accuracy is very important in order to obtain a statistically significant and unambiguous match This method is best applied to completely sequenced genomes and well characterised proteomes. However, depending on the data quality, proteins that are highly homologous to already characterised proteins (greater than 80 to 90% sequence identity) can also be identified. The retrieved sequence are evaluated by mass accuracy of the peptides, matching of additional peptide masses in the MALDI spectrum after accounting for common modifications such as oxidation, acrylamidation of cysteine and missed cleavages and the use of secondary information (apparent isoelectric point and molecular weight). If any ambiguity about the identification by MALDI-MS still exists, the results must verified by an other identification method. Peptide mass fingerprint is generally carried out with a MALDI-TOF (Matrix-assisted laser desorption ionization time-of-flight ) instrument but can also be achieved ESI-TOF (Electrospray Ionisation time-of-flight) or LC-MS (Liquid Chromatography-Mass Spectrometry) mass spectrometer.
<tr><td>definition_reference:<td> PMID:10967324
<tr><td>definition_reference:<td> PMID:11752590
<tr><td>definition_reference:<td> PMID for application instance:11805826
<tr><td colspan=2><hr><tr><tr><td>term:<td> peptide synthesis
<tr><td>goid:<td> <a name="MI:0083">MI:0083
<tr><td>definition:<td> When one of the partners participates in the interaction with a relatively short peptide fragment, it is often convenient to precisely identify the minimal region that supports the interaction by synthesising a series of overlapping peptides and by testing them in the binding assay. Synthetic peptides that are identical with peptides synthesised in vivo are useful experimental tools for such studies. Peptides are routinely synthesised in a test tube from monomeric amino acids by condensation reactions that form peptide bonds. Peptides are constructed sequentially by coupling the C-terminus of a monomeric amino acid to the N-terminus of the growing peptide. To prevent unwanted reactions involving the amino groups and carboxyl groups of the side chains during the coupling steps, a protecting (blocking) group is attached to the side chains. Without these protecting groups, branched peptides would be generated. In the last steps of synthesis, the side chain-protecting groups are removed and the peptide is cleaved from the resin on which synthesis occurs. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> phage display
<tr><td>goid:<td> <a name="MI:0084">MI:0084
<tr><td>definition:<td> Peptide sequences or entire proteins can be displayed on phage capsids by fusion to coat proteins to generate a library of fusion phages each displaying a different peptide. Such a library can then be exploited to identify specific phages that display peptides that bind to any given bait molecule for instance an antibody. The selection is performed by a series of cycles of affinity purification known as panning. The bait protein, immobilized on a solid support (plastic, agarose, sepharose, magnetic beads...) is soaked in the phage mixture and that phage that remains attached to the bait is amplified and carried through a further affinity purification step. Each cycle results in an approximately 1,000-fold enrichment of specific phage and after a few selection rounds (2-4), DNA sequencing of the tight-binding phage reveals only a small number of sequences. Phage display panning experiments can be carried out either on libraries of peptides of random amino acid sequence or on libraries of displaying natural peptides obtained by inserting cDNA fragments into the phage vector (cDNA libraries). Libraries have been assembled on several different phages (Fd, Lambda, T7).
<tr><td>definition_reference:<td> PMID:7708014
<tr><td>definition_reference:<td> PMID:10975452
<tr><td colspan=2><hr><tr><tr><td>term:<td> phosphorylation
<tr><td>goid:<td> <a name="MI:0170">MI:0170
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> phylogenetic profile
<tr><td>goid:<td> <a name="MI:0085">MI:0085
<tr><td>definition:<td> The phylogenetic profile of a protein stores information about the presence and the absence of that protein in a set of genomes. By clustering identical or similar profiles, proteins with similar functions and potentially interacting are identified.
<tr><td>definition_reference:<td> PMID:10200254
<tr><td colspan=2><hr><tr><tr><td>term:<td> pisa: protein in situ array
<tr><td>goid:<td> <a name="MI:0092">MI:0092
<tr><td>definition:<td> Protein In Situ Array is a method by which protein arrays are rapidly generated in one step directly from DNA, by cell-free protein expression and simultaneous in situ immobilisation at a surface. Individual genes or fragments are produce by PCR or RT-PCR depending on the source of genetic material using properly designed primers. The PISA is generated by cell-free protein synthesis using coupled transcription and translation to produce a double HexaHis-tagged protein, the reaction being carried out on a surface to which the protein adheres as soon as it is synthesised. 
<tr><td>definition_reference:<td> PMID:11470888
<tr><td colspan=2><hr><tr><tr><td>term:<td> polyclonal antibody
<tr><td>goid:<td> <a name="MI:0086">MI:0086
<tr><td>definition:<td> Polyclonal antibodies are a mixture of different antibodies that represent the immune response, normally in an experimental animal, to any given antigen.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> predictive text mining
<tr><td>goid:<td> <a name="MI:0087">MI:0087
<tr><td>definition:<td> Methods based on natural language processing to detect possible interactions between proteins (direct physical interactions or indirect genetic interactions). This includes the detection of non ambiguous protein or gene names and analysis of the relation expressed in a sentence among them.
<tr><td>definition_reference:<td> PMID:11791231
<tr><td colspan=2><hr><tr><tr><td>term:<td> primer specific pcr
<tr><td>goid:<td> <a name="MI:0088">MI:0088
<tr><td>definition:<td> Sequences can be identified in a DNA mixture by launching a PCR (Polymerase Chain Reaction) controlled by sequence specific primers. Such reaction starts only when the hybridization of the primer with a complementary sequence occurs.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> protein array
<tr><td>goid:<td> <a name="MI:0089">MI:0089
<tr><td>definition:<td> The protein array technology allows the screening of biochemical activities or binding abilities of hundreds or thousands of protein samples in parallel. After synthesis and purification by high-throughput methodologies, the proteins are printed onto the chip by using an instrument (micro-arrayer) that is capable of spotting liquid samples in a reproducible manner onto a planar support. The ordered protein array can then be probed with labelled molecules to identify proteins that bind to the bait.
<tr><td>definition_reference:<td> PMID:12067604
<tr><td>definition_reference:<td> PMID:10976071
<tr><td colspan=2><hr><tr><tr><td>term:<td> protein sequence identification
<tr><td>goid:<td> <a name="MI:0093">MI:0093
<tr><td>definition:<td> Single amino acid identification along a protein sequence. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> protein staining
<tr><td>goid:<td> <a name="MI:0094">MI:0094
<tr><td>definition:<td> A wide range of dyes have been used over the years to visualise proteins in polyacrylamide gels - Coomasie Blue and silver-staining being two classical methods. Fluorescent dyes such as Nile Red and SYPRO Orange are now increasingly used due to their superior dynamic range. Use of non-denaturing gels can allow visualisation of protein protein interactions. Several dyes can be used to specifically indicate residue modification, however this methodology will give no information as the number of residues modified or their position within the protein sequence. Examples include the use of acid fuscian or the fluorescent dansyl hydrazine to show protein glycosylation.
<tr><td>definition_reference:<td> PMID:12015990
<tr><td colspan=2><hr><tr><tr><td>term:<td> ptm: post translation modification
<tr><td>goid:<td> <a name="MI:0120">MI:0120
<tr><td>definition:<td> Residue covalent modifications occurring in the specific protein form involved in an interaction 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> pull down
<tr><td>goid:<td> <a name="MI:0096">MI:0096
<tr><td>definition:<td> A specific affinity chromatography method where a protein of interest (bait) is expressed as a fusion to an affinity tag (GST, HIS tail ...) and linked at high concentration to a support that has affinity for the tag. Purified proteins or cellular extracts are then adsorbed to the resin and the retained binding proteins are identified. Thus, in this approach, the protein that has affinity for the solid support (bait) is expressed and purified first, often in an etherologous system, and then challenged with a solution containing the candidate partner proteins. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> reverse rrs: reverse ras recruitment system
<tr><td>goid:<td> <a name="MI:0097">MI:0097
<tr><td>definition:<td> In this complementation approach the bait can be any membrane protein (for example a receptor or a channel protein), the prey is cloned as a fusion protein of any cDNA from a library and the coding sequence of cytoplasmic RAS (cdc25 in yeast). If the bait and the prey interact, RAS is recruited close to the membrane and can activate cell growth. This procedure must take place in cells having a mutated RAS (Cdc25-2 yeast strain having a temperature sensitive mutation of RAS) to avoid constitutive growth activation.
<tr><td>definition_reference:<td> PMID:11160938
<tr><td colspan=2><hr><tr><tr><td>term:<td> ribosome display
<tr><td>goid:<td> <a name="MI:0098">MI:0098
<tr><td>definition:<td> This method permits the coupling of phenotype to genotype via the formation of a non-covalent ternary complex between mRNAs and their encoded polypeptides while they are translated in an in vitro system. As a first step a cDNA library is constructed that encodes chimeric proteins in which the natural proteins or protein domains are fused to a C-terminal tether. As a consequence when the mRNA is translated in vitro the domain can fold while the tether is still in the ribosomal tunnel. Furthermore this chimeric mRNAs lack a stop codon, thus preventing release of the mRNA and the polypeptide from the ribosome. High concentrations of magnesium and low temperature further stabilise the ternary complex. Similarly to phage display, these complexes can be used directly to select for nucleic acids encoding proteins with desired properties.
<tr><td>definition_reference:<td> PMID:11551470
<tr><td>definition_reference:<td> PMID for application instance:12167034
<tr><td colspan=2><hr><tr><tr><td>term:<td> S-farnesyl-L-cysteine
<tr><td>goid:<td> <a name="MI:0151">MI:0151
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0102
<tr><td colspan=2><hr><tr><tr><td>term:<td> S-geranylgeranyl-L-cysteine
<tr><td>goid:<td> <a name="MI:0152">MI:0152
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0104
<tr><td colspan=2><hr><tr><tr><td>term:<td> S-palmitoyl-L-cysteine
<tr><td>goid:<td> <a name="MI:0154">MI:0154
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0106
<tr><td colspan=2><hr><tr><tr><td>term:<td> S-phospho-L-cysteine
<tr><td>goid:<td> <a name="MI:0173">MI:0173
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td>definition_reference:<td> RESID:AA0034
<tr><td colspan=2><hr><tr><tr><td>term:<td> seldi chip: proteinchip(r) on a surface-enhanced laser desorption/ionization
<tr><td>goid:<td> <a name="MI:0095">MI:0095
<tr><td>definition:<td> ProteinChip(r) Array technology is a surface-enhanced laser desorption/ionization (SELDI) approach (Ciphergen Biosystems Inc. Fremont, CA, USA) for sample fractionation accomplished by retentate chromatography. Retentate chromatography is performed on ProteinChip Arrays with varying chromatographic properties (e.g. anion exchange, cation exchange, metal affinity and reverse phase). By utilising arrays with differing surface chemistries in parallel and in series, a complex mixture of proteins, as from cells or body fluids, can be resolved into subsets of proteins with common properties. Specific analytes can also be examined by using preactivated arrays to which a bait molecule (such as an antibody or biotinylated DNA) is immobilized and a solution containing the binding partner(s) is presented to the array. This array-based immunoprecipitation or protein-binding experiment has been used with good success to study DNA-binding proteins, receptor-ligand interactions, and protein complexes. Any ligand retained on a SELDI chip can directly be identified by mass spectrometry.
<tr><td>definition_reference:<td> PMID:11827829
<tr><td colspan=2><hr><tr><tr><td>term:<td> sequence based phylogenetic profile
<tr><td>goid:<td> <a name="MI:0100">MI:0100
<tr><td>definition:<td> Multiple alignments of orthologous sequences in the same species and their corresponding phylogenetic trees are built. Every phylogenetic tree is computed as a matrix of distances between all possible protein pairs. The covariation of the distance matrices reveals interacting protein pairs.
<tr><td>definition_reference:<td> PMID:11707606
<tr><td colspan=2><hr><tr><tr><td>term:<td> sequence based prediction
<tr><td>goid:<td> <a name="MI:0101">MI:0101
<tr><td>definition:<td> Computational methods based on evolutionary hypothesis, used as criteria to browse sequences and predict interacting pairs
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> sequence tag identification
<tr><td>goid:<td> <a name="MI:0102">MI:0102
<tr><td>definition:<td> This approach leads to protein identification by combining mass measurement and short amino acid sequence information obtained by tandem mass spectrometry. This information is then used to automatically find the best match in a sequence database. A mixture of peptides derived from a protease digestion is analysed by nanoelectrospray LC-MS/MS (Liquid Chromatography Tandem Mass Spectrometer or nanoESI MS/MS) mass spectrometry. Electrospray mass spectrometry cannot be applied to dilute samples and is affected by high salt. As a consequence peptides, normally extracted from acrylamide gels by in situ proteolysis, are desalted and concentrated on a microcolumn followed by elution into a capillary used for nanoelectrospray tandem mass spectrometry. A first mass spectrum ("Normal mass spectrum" or "Q1 mass spectrum") gives information about the masses of all the peptides. Peptides observed in the normal mass spectrum are isolated in turn and dissociated into fragments by collision with gas molecules within the mass spectrometer. Some of the fragments obtained from a peptide constitute a nested set, differing by one amino acid, and the mass difference between them allows assignment of a partial sequence. The masses of the fragments define the position of the partial sequence in the peptide. Together with the cleavage specificity of the protease used to cleave the protein, and mass information such sequence tag provides much higher search specificity to match the a database entry. The procedure is repeated with several peptides from the digest, resulting in multiple identifications of the same protein or identification of several proteins from the peptide mixture. Unknown proteins can easily be identified by using the high specificity of the peptide sequence tag for searches in most sequence databases including EST or genome databases.
<tr><td>definition_reference:<td> PMID:10967324
<tr><td>definition_reference:<td> PMID:11752590
<tr><td>definition_reference:<td> PMID for application instance:11805837
<tr><td colspan=2><hr><tr><tr><td>term:<td> southern blot
<tr><td>goid:<td> <a name="MI:0103">MI:0103
<tr><td>definition:<td> A standard procedure to identify DNA fragments containing specific gene sequences. In this procedure i) a genome is fragmented using a restriction enzyme ii) the generated fragments are separated by electrophoresis iii) the fragments are transferred to a membrane iv)the membrane is incubated with a radio labelled probe that hybridises any complementary subsequence.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> spa: scintillation proximity assay
<tr><td>goid:<td> <a name="MI:0099">MI:0099
<tr><td>definition:<td> SPA relies upon the fact that a beta particle emitted from a radioisotope decay can excite a fluorophore only when its at a very short distance in water solution (few micrometers). The ligand is labelled with a radioactive atom and its potential partner is fixed to fluorophore containing beads, the emitted fluorescence proving their interaction can be measured in a scintillation counter. The scintillator measures only the amount of bound radiolabelled ligand. Competition experiment with cold competitor can be done to estimate the binding affinities (50% inhibitory concentration [IC50], cold ligand versus labelled ligand). Loss of signal can also be used to measure substrate cleavage by an enzyme, and labelled antibodies used to titrate the degree of modified residue present.
<tr><td>definition_reference:<td> PMID:3866247
<tr><td colspan=2><hr><tr><tr><td>term:<td> static light scattering
<tr><td>goid:<td> <a name="MI:0104">MI:0104
<tr><td>definition:<td> In static light scattering, the average intensity of scattered light at multiple angles is measured. The data yield information on particle molecular weight, particle size and shape, and particle-particle interactions.
<tr><td>definition_reference:<td> PMID:9013660
<tr><td colspan=2><hr><tr><tr><td>term:<td> structure based prediction
<tr><td>goid:<td> <a name="MI:0105">MI:0105
<tr><td>definition:<td> Methods based on 3D structure information.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> surface patches
<tr><td>goid:<td> <a name="MI:0106">MI:0106
<tr><td>definition:<td> Surface patches are built using 6 criteria: solvation potential, residue interface propensity, hydrophobicity, planarity, protrusion and accessible surface area. Protein structures having similar patches are likely to have the same interactions.
<tr><td>definition_reference:<td> PMID:9299343
<tr><td colspan=2><hr><tr><tr><td>term:<td> surface plasmon resonance
<tr><td>goid:<td> <a name="MI:0107">MI:0107
<tr><td>definition:<td> This method measures formation of complex by monitoring changes in the resonance angle of light impinging on a gold surface as a result of changes in the refractive index of the surface. A ligand of interest (peptide or protein) is immobilized on a dextran polymer, and a solution of interacting protein is passed over it through a cell, with a gold wall coated with this polymer. Macromolecules that interact with the immobilized ligand are retained on the polymer surface, and alter the resonance angle of impinging light as a result of the change in refractive index brought about by the increased protein mass retained on the polymer surface. Since all proteins have the same refractive index and since there is a linear correlation between resonance angle shift and protein concentration near the surface, this allows one to measure changes in protein concentration at the surface as a consequence of protein interaction. Furthermore, this can be done in real time, allowing direct measurement of both the on rate and the off rate of complex formation. 
<tr><td>definition_reference:<td> PMID:12120258
<tr><td>definition_reference:<td> PMID:11896282
<tr><td colspan=2><hr><tr><tr><td>term:<td> t7 phage display
<tr><td>goid:<td> <a name="MI:0108">MI:0108
<tr><td>definition:<td> T7 is a double stranded DNA bacteriophage with a thin-walled icosahedral capsid, ~55 nm in diameter, which is decorated by 415 copies of the capsid protein, the product of gene 10. gp10 can tolerate insertions at the carboxyterminus without loosing its ability to be inserted into functional phage capsids. Both low density and high density display (albeit only with short peptides) can be achieved.
<tr><td>definition_reference:<td> PMID:-
<tr><td>comment:<td> reference not index in medline : Rosenberg, A., Griffin, K., Studier, W.S., McCormick, M., Berg, J., Novy, R., Mierendorf, R. inNovations, 1996, 6, 1.
<tr><td colspan=2><hr><tr><tr><td>term:<td> tap tag coip: tap tag coimmunoprecipitation
<tr><td>goid:<td> <a name="MI:0109">MI:0109
<tr><td>definition:<td> The TAP method involves the fusion of the TAP tag (encoding a calmodulin binding peptide, a TEV cleavage site, and the Staphylococcus aureus Protein A) to the target protein and the introduction of the construct into the host cell or organism, maintaining the expression of the fusion protein at, or close to, its natural level. The fusion protein and associated components are recovered from cell extracts by affinity selection on an IgG matrix. After washing, the TEV protease is added to release the bound material. The eluate is incubated with calmodulin-coated beads in the presence of calcium. This second affinity step is required to remove the TEV protease as well as traces of contaminants remaining after the first affinity selection. After washing, the bound material is released with EGTA. This two steps purification steps ensures a highly selective complex purification of the "Tapped" protein (first round of selection on the protein A, a high affinity tag) under mild condition (non denaturant pH or conditions required to remove the tag). 
<tr><td>definition_reference:<td> PMID:10504710
<tr><td colspan=2><hr><tr><tr><td>term:<td> text mining
<tr><td>goid:<td> <a name="MI:0110">MI:0110
<tr><td>definition:<td> Text mining methods can be used to predict or confirm interactions by automated processing of scientific literature. Co-occurence in the same sentence of an abstract of gene products labels are analysed to evaluate whether it represents a valid evidence of an interaction. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> two hybrid
<tr><td>goid:<td> <a name="MI:0018">MI:0018
<tr><td>definition:<td> The classical yeast two-hybrid system is a method that uses transcriptional activity as a measure of protein-protein interaction. It relies on the modular nature of many site-specific transcriptional activators (GAL 4) , which consist of a DNA-binding domain and a transcriptional activation domain. The DNA-binding domain serves to target the activator to the specific genes that will be expressed, and the activation domain contacts other proteins of the transcriptional machinery to enable transcription to occur. The two-hybrid system is based on the observation that the two domains of the activator need to be non-covalently brought together by the interaction of any two proteins. The application of this system requires the expression of two hybrid
<tr><td>definition_reference:<td> PMID:10967325
<tr><td>definition_reference:<td> PMID:12634794
<tr><td colspan=2><hr><tr><tr><td>term:<td> two hybrid array
<tr><td>goid:<td> <a name="MI:0397">MI:0397
<tr><td>definition:<td> Two-hybrid screening can be done in a colony array format, in which each colony expresses a defined pair of proteins. Because the particular protein pair expressed in each colony is defined by its position in the array, positive signals identify interacting proteins without further characterization, thus obviating the need for DNA purification and sequencing. The interrogation of a two-hybrid colony array usually involves a mating strategy in which every DNAbinding domain hybrid (the bait) is tested against all activation domain hybrids (thepreys) in a grid pattern. Arrays usually use full-length open reading frames.\n\n
<tr><td>definition_reference:<td> PMID:11827624
<tr><td>definition_reference:<td> PMID for application instance:10688190
<tr><td colspan=2><hr><tr><tr><td>term:<td> two hybrid fragment pooling approach
<tr><td>goid:<td> <a name="MI:0399">MI:0399
<tr><td>definition:<td> This two hybrid approac involve the screening of a large number of individual proteins against a comprehensive library of randomly generated fragment as prey. The usage of degenarated fragment allow identification of the minimal protein region required for the interaction. since multiple clones that encode overlapping regions of protein are often identified, the minimal domain for interaction may be readily apparent from the initial screen.
<tr><td>definition_reference:<td> PMID:12634794
<tr><td>definition_reference:<td> PMID for application instance:11196647
<tr><td colspan=2><hr><tr><tr><td>term:<td> two hybrid pooling approach
<tr><td>goid:<td> <a name="MI:0398">MI:0398
<tr><td>definition:<td> In the pooling strategy the sets of bait and prey hybrid vectors are mated all together and then selected. The positives double hybrid clones are sequenced to identify the interacting partners. 
<tr><td>definition_reference:<td> PMID:12634794
<tr><td>definition_reference:<td> PMID for application instance:11283351
<tr><td colspan=2><hr><tr><tr><td>term:<td> ubiquitin reconstruction
<tr><td>goid:<td> <a name="MI:0112">MI:0112
<tr><td>definition:<td> In this method the two proteins, whose interaction is under investigation, (in this case mostly membrane proteins) are fused to an terminal fragment (Nub) and to a C-terminal fragment of ubiquitin (Cub). The two fragments do not associate to form a functional ubiquitin unless the two fused membrane proteins form a complex. The association is monitored by an ingenious trick. The C-term fragment of ubiquitin is expressed as a fusion to a transcription factor that being linked to a membrane protein cannot perform its function unless it is released form the ubiquitin fusion by a specific protease. This achieved through the activity of UBP (an ubiquitin specific protease) that cleaves off the reporter protein only when a functional ubiquitin is reconstituted.
<tr><td>definition_reference:<td> PMID:9560251
<tr><td colspan=2><hr><tr><tr><td>term:<td> ubiquitinated
<tr><td>goid:<td> <a name="MI:0189">MI:0189
<tr><td>definition:<td> residue modification
<tr><td>definition_reference:<td> PMID:11125103
<tr><td colspan=2><hr><tr><tr><td>term:<td> undetermined participant
<tr><td>goid:<td> <a name="MI:0396">MI:0396
<tr><td>definition:<td> Molecule whose sequence identity is not checked and has been assumed from external or previous experimental evidence(s).
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> western blot
<tr><td>goid:<td> <a name="MI:0113">MI:0113
<tr><td>definition:<td> Western blot is a procedure to identify and quantify proteins. A mixture of protein is first submitted to an electrophoresis in denaturant condition and then electro-transferred from the gel to a membrane. The membrane is then incubated with a primary antibody specific for a given protein or a specific residue modification in the sample under analysis. A secondary antibody, radiolabelled or fused to a chromogenic enzyme, targets the first antibody and allows the visualisation of the protein band on the membrane. 
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> x-ray: x-ray crystallography
<tr><td>goid:<td> <a name="MI:0114">MI:0114
<tr><td>definition:<td> X-rays have a wavelength, typically around 0.1 nm (the diameter of a hydrogen atom). If a narrow parallel beam of X-rays is directed at a sample of a pure protein, most of the X-rays will pass straight through it. A small fraction, however, will be scattered by the atoms in the sample. If the sample is a well-ordered crystal, the scattered waves will reinforce one another at certain points and will appear as diffraction spots when the X-rays are recorded by a suitable detector. The position and intensity of each spot in the X-ray diffraction pattern contain information about the position and nature of the atoms in the crystal. The three-dimensional structure of a large molecule can be deduced from the electron-density map of its crystal. In recent years X-ray diffraction analysis has become increasingly automated, and now the slowest step is likely to be the production of suitable protein crystals. This requires high concentration of very pure protein and empirical searching for the proper crystallisation conditions.
<tr><td>definition_reference:<td> PMID:-
<tr><td colspan=2><hr><tr><tr><td>term:<td> yeast display
<tr><td>goid:<td> <a name="MI:0115">MI:0115
<tr><td>definition:<td> The proteins are displayed on the surface of the yeast S. cerevisiae by fusion to signal sequences for protein secretion. This method is limited by the low efficiency of the yeast display system but can take full advantage of the possibility of exploiting cell sorting methods (FACS) to isolate cells that display molecules with desired binding properties.
<tr><td>definition_reference:<td> PMID:9181578
<tr><td colspan=2><hr><tr></table></html>